<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="7f520d13-6627-43a5-8ee8-f1d1f217517d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";journal:journal:pnas;page:string:Article/Chapter View;requestedJournal:journal:pnas;ctype:string:Journal Content;website:website:pnas-site;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;article:article:doi\:10.1073/pnas.0501761102;subPage:string:Full Text;issue:issue:doi\:10.1073/pnas.2005.102.issue-15"/>
<script>
    PNASdataLayer ={"ecommerce":{},"page":{"pageInfo":{"nlmArticleType":"research-article","issue":"15","pageTitle":"Functional genomic analysis of the rates of protein evolution | Proceedings of the National Academy of Sciences","author":"Dennis P. Wall|Aaron E. Hirsh|Hunter B. Fraser|Jochen Kumm|Guri Giaever|Michael B. Eisen|Marcus W. Feldman","pubDate":"2005-03-30","seriesTitle":"pnas","volume":"102","pageType":"article","articleType":"Research Article","viewType":"full","pageURL":"https://www.pnas.org/doi/10.1073/pnas.0501761102","issueDate":"2005-04-12","DOI":"10.1073/pnas.0501761102"},"attributes":{"accessType":"Free","licenseType":"PNAS","keywords":["protein fitness","protein dispensability","structural equation models"],"freeAccess":"yes","openAccess":"no","sp-feat":"","colloquia":"","topic":"Evolution"}},"user":{"instType":"","geo":"United States","institution":"CLAREMONT COLLEGES PERIODICALS DEPT","registeredUser":"no","authenticated":"yes","access":"yes","institutionID":513616,"groups":["Unregistered no accesses","All users group","All institution users group","Known Users","All users no webcrawlers","All ip range users group"]}};
    if (PNASdataLayer) {
        var ua = navigator.userAgent;
        var deviceType = "desktop";
        var width = window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight;
        var hight = window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth;
        if (/(tablet|ipad|playbook|silk)|(android(?!.*mobi))/i.test(ua)) deviceType = "etc";
        if (/Mobile|iP(hone|od)|Android|BlackBerry|IEMobile|Kindle|Silk-Accelerated|(hpw|web)OS|Opera M(obi|ini)/.test(ua)) deviceType = "mobile";
        PNASdataLayer.site = {};
        PNASdataLayer.site.userAgent = ua;
        PNASdataLayer.site.device = deviceType;
        PNASdataLayer.site.viewport = width + "x" + hight;
    }

    window.dataLayer = window.dataLayer || []
    window.dataLayer.push({
        'event': 'get-meta-data',
        'pnasDataLayer': PNASdataLayer
    })
</script>

<meta charset="UTF-8">
<meta name="robots" content="noarchive"/>




















    















    <meta property="og:title" content="Functional genomic analysis of the rates of protein evolution"/>
    <meta property="og:type" content="Article">
    <meta property="og:url" content="https://www.pnas.org/doi/abs/10.1073/pnas.0501761102">
    <meta property="og:image" content="https://www.pnas.org/cms/asset/5e670e3c-57dd-4955-b66f-675d06d9e5ed/pnas.2005.102.issue-15.cover.gif"/>
    <meta property="og:site_name" content="PNAS"/>
    <meta property="og:description" content="The evolutionary rates of proteins vary over several orders of magnitude. Recent work
suggests that analysis of large data sets of evolutionary rates ..."/>

<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=0"/>
<title>Functional genomic analysis of the rates of protein evolution | PNAS</title>










            <link rel="stylesheet" type="text/css" href="/wro/l7ul~article-metrics-phase2.css">
        



    
    
                <link rel="stylesheet" href="/products/pnas/releasedAssets/css/build-e41ad7f57abf398b554b.css" media="all" id="build-style-sheet">
                <script>
                    function isIE() { //won't check for edge browser
                        var ua = navigator.userAgent;
                        /* MSIE used to detect old browsers and Trident used to newer ones*/
                        var is_ie = ua.indexOf("MSIE ") > -1 || ua.indexOf("Trident/") > -1;
          
                        return is_ie; 
                    }
                    
                    if(isIE()) {
                        document.write('<link rel="stylesheet" href="/products/pnas/releasedAssets/css/build-style-fallback-860c47ec19da799ba0a9.css" media="all" />')
                        var buildCss = document.getElementById('build-style-sheet');
                        buildCss.parentNode.removeChild(buildCss);
                    }
                </script>
        <link rel="stylesheet" href="/products/pnas/releasedAssets/css/print-e41ad7f57abf398b554b.css"media="print">
    
        <script>
        try {
            document.documentElement.classList.add('js');
        } catch (e) {
        }
    </script>
     
    





<meta http-equiv="X-UA-Compatible" content="IE=edge">
<script type="text/javascript">var addthis_config={data_track_clickback:false}</script>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="google-site-verification" content="VqWyHrlr3c-0idRWihMWQbguIlkFuw04eNaZwk8-6PY" />
<meta name="google-site-verification" content="8gjxFs4TwHihlvJmGwrCnm3HI3XMRdORqlpfTyj6nCs" />
<meta name="msvalidate.01" content="5C8EE388274E2EEF3889410100369D50" />
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/pnas-styles-1651154811397.css"><script defer src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon='{"rayId":"6e65fb942de2942c","token":"4fddf4ff2db74c469c27e6f3ddde2174","version":"2021.12.0","si":100}' crossorigin="anonymous"></script>
<script defer src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon='{"rayId":"6e9db4c0dd526450","token":"4fddf4ff2db74c469c27e6f3ddde2174","version":"2021.12.0","si":100}' crossorigin="anonymous"></script>

            <link rel="canonical" href="https://www.pnas.org/doi/10.1073/pnas.0501761102">
        





    <script src="//scholar.google.com/scholar_js/casa.js" async></script>

    <!-- Google Tag Manager -->
    <script>
    var dataLayer = dataLayer ||[];
</script>


    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-KCV32QR');</script>
    
    <!-- Google Tag Manager (noscript) -->
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KCV32QR" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <!-- End Google Tag Manager (noscript) -->

    <!-- End Google Tag Manager -->
</head>
<body class="pb-ui">



















<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>





    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">




        
        <div>



        
        <header class="main-header"><a href="#main" class="sr-only sr-only-focusable skip-to-main-content serif font-weight-bold bg-white p-2 text-black-gray text-decoration-none">Skip to main content</a><nav id="mainNavbar" aria-label="main navigation" class="main-navbar navbar bg-white navbar-light navbar-expand-xl text-uppercase"><div class="container-fluid"><div class="navbar-in">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="729a0980-ae74-4b8b-8362-07d931f05f0e" class="navbar-in">
        



        
        <a href="/" class="navbar-brand navbar__logo--default">
<img src="/pb-assets/images/Logos/header-logo/logo-1624644560537.svg" title="PNAS logo" alt="PNAS Logo">
</a>
<a href="/" class="navbar-brand navbar__logo--light">
<img src="/pb-assets/images/Logos/header-logo/logo-light-1641218818190.svg" title="PNAS logo" alt="PNAS Logo">
</a>

        </div>
    
<div class="navbar-toggler__container">



        
        <a href="#" aria-expanded="false" aria-label="Expand search field" aria-controls="quick-search-form-0" data-expand-label="Expand search field" data-collapse-label="Collapse search field" data-target=".header-quick-search" data-toggle="collapse" class="btn quick-search__trigger"><svg xmlns="http://www.w3.org/2000/svg" width="45" height="45" viewBox="0 -8 32 50" class="quick-search-icon-animated"><path d="M17.71,17.91a6.25,6.25,0,1,1,1.89-4.48,6.23,6.23,0,0,1-1.89,4.48" class="cls cls-1"></path><line x1="25.07" y1="25.15" x2="8.2" y2="8.29" class="cls cls-2"></line><line x1="8.2" y1="25.15" x2="25.07" y2="8.29" class="cls cls-3"></line></svg></a>
<button type="button" data-slider="side-menu" data-action="toggle" aria-controls="#slider-side-menu" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler__box"><span class="navbar-toggler__inner"></span></span></button></div></div>



        
        <div id="main-menu" class="main-menu collapse navbar-collapse collapse-lock-screen"><div class="main-menu__nav flex-grow-1 d-flex justify-content-center"><ul class="navbar-nav"><li class="nav-item px-3"><div class="dropdown"><a id="menu-item-0" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="nav-link dropdown-toggle">ARTICLES</a><div aria-labelledby="menu-item-0" class="dropdown-menu"><a href="/toc/pnas/current" class="dropdown-item">Current Issue</a><a href="/latest" class="dropdown-item">Latest Articles</a><a href="/about/special-features" class="dropdown-item">Special Features</a><a href="/loi/pnas" class="dropdown-item">List of Issues</a><a href="https://academic.oup.com/pnasnexus" class="dropdown-item dropdown-item--external"><i>PNAS Nexus</i></a></div></div></li><li class="nav-item px-3"><a id="menu-item-1" href="/front-matter" tabindex="" class="nav-link">Front Matter</a></li><li class="nav-item px-3"><div class="dropdown"><a id="menu-item-2" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="nav-link dropdown-toggle">AUTHORS</a><div aria-labelledby="menu-item-2" class="dropdown-menu"><a href="/author-center" class="dropdown-item">Information for Authors</a><a href="/author-center/editorial-and-journal-policies" class="dropdown-item">Editorial and Journal Policies</a><a href="/author-center/submitting-your-manuscript" class="dropdown-item">Submission Procedures</a><a href="/author-center/publication-charges" class="dropdown-item">Publication Charges</a></div></div></li><li class="nav-item px-3"><div class="topicsButton">



        
        <a data-target=".topics-container" data-toggle="collapse" class="btn btn-outline-primary btn-topics">Topics
    <i aria-hidden="true" class="icon-plus"></i>
    <i aria-hidden="true" class="icon-close"></i>
</a>
  
<div id="topics" class="topics-container collapse">









        <div data-widget-def="UX3Tabs" data-widget-id="e657c77c-421c-46e0-b0a0-d72b63228a8b" class="topics-tab text-capitalize">
        



        
        <ul role="tablist" class="nav nav-tabs"><li role="presentation" class="nav-item"><a id="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-physical-sciences-tab" data-toggle="tab" href="#nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-physical-sciences-pane" role="tab" aria-controls="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-physical-sciences-pane" aria-selected="true" class="nav-link  active"><span>Physical Sciences</span></a></li><li role="presentation" class="nav-item"><a id="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-social-sciences-tab" data-toggle="tab" href="#nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-social-sciences-pane" role="tab" aria-controls="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-social-sciences-pane" aria-selected="false" class="nav-link"><span>Social Sciences</span></a></li><li role="presentation" class="nav-item"><a id="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-biological-sciences-tab" data-toggle="tab" href="#nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-biological-sciences-pane" role="tab" aria-controls="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-biological-sciences-pane" aria-selected="false" class="nav-link"><span>Biological Sciences</span></a></li></ul><div class="tab-content"><div id="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-physical-sciences-pane" aria-labelledby="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-physical-sciences-tab" role="tabpanel" class="tab-pane fade show active">



        
        <div class="topics-tab__topic">
    <div class="topics-tab__topic__portal">
      <h4 class="topic__heading">Featured Topics</h4>
      <ul class="rli topic__items topic__section">
        <li class="topic__item">
            <a href="/topic/phys">Physics</a>
        </li>
        <li class="topic__item">
            <a href="/topic/chem">Chemistry</a>
        </li>
        <li class="topic__item">
            <a href="/sustainability-science">Sustainability Science</a>
        </li>
      </ul>
    </div>
    <div class="topics-tab__topic__article">
      <h4 class="topic__heading">Articles By Topic</h4>
      <div class="d-flex">
        <ul class="rlist topic__items topic__section">
          <li class="topic__item">
            <a href="/topic/app-math">Applied Mathematics</a>
          </li>
          <li class="topic__item">
            <a href="/topic/app-phys">Applied Physical Sciences</a>
          </li>
          <li class="topic__item">
            <a href="/topic/astro">Astronomy</a>
          </li>
          <li class="topic__item">
            <a href="/biophysics-and-computational-biology">Biophysics and Computational Biology</a>
          </li>
          <li class="topic__item">
            <a href="/topic/comp-sci">Computer Sciences</a>
          </li>
        </ul>
        <ul class="rlist topic__items topic__section">
            <li class="topic__item">
                <a href="/topic/earth-sci">Earth, Atmospheric, and Planetary Sciences</a>
            </li>
            <li class="topic__item">
                <a href="/topic/eng">Engineering</a>
            </li>
            <li class="topic__item">
                <a href="/environmental-sciences">Environmental Sciences</a>
            </li>
            <li class="topic__item">
                <a href="/topic/math">Mathematics</a>
            </li>
            <li class="topic__item">
                <a href="/topic/stat">Statistics</a>
            </li>
        </ul>
      </div>
    </div>
</div>
</div><div id="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-social-sciences-pane" aria-labelledby="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-social-sciences-tab" role="tabpanel" class="tab-pane fade">



        
        <div class="topics-tab__topic">
    <div class="topics-tab__topic__portal">
        <h4 class="topic__heading">Featured Topics</h4>
            <ul class="rli topic__items topic__section">
                <li class="topic__item">
                    <a href="/anthropology">Anthropology</a>
                </li>
                <li class="topic__item">
                    <a href="/sustainability-science">Sustainability Science</a>
                </li>
            </ul>
    </div>
    <div class="topics-tab__topic__article">
        <h4 class="topic__heading">Articles By Topic</h4>
            <div class="d-flex">
                <ul class="rlist topic__items topic__section">
                    <li class="topic__item">
                        <a href="/topic/econ-sci">Economic Sciences</a>
                    </li>
                    <li class="topic__item">
                        <a href="/environmental-sciences">Environmental Sciences</a>
                    </li>
                </ul>
                <ul class="rlist topic__items topic__section">
                    <li class="topic__item">
                        <a href="/topic/pol-sci">Political Sciences</a>
                    </li>
                    <li class="topic__item">
                        <a href="/psychological-and-cognitive-sciences">Psychological and Cognitive Sciences</a>
                    </li>
                </ul>
            </div>
    </div>
</div>
</div><div id="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-biological-sciences-pane" aria-labelledby="nav-e657c77c-421c-46e0-b0a0-d72b63228a8b-biological-sciences-tab" role="tabpanel" class="tab-pane fade">



        
        <div class="topics-tab__topic">
    <div class="topics-tab__topic__portal">
        <h4 class="topic__heading">Featured Topic</h4>
            <ul class="rli topic__items topic__section">
                <li class="topic__item">
                    <a href="/sustainability-science">Sustainability Science</a>
                </li>
            </ul>
    </div>
    <div class="topics-tab__topic__article">
        <h4 class="topic__heading">Articles By Topic</h4>
            <div class="d-flex">
                <ul class="rlist topic__items topic__section">
                    <li class="topic__item">
                        <a href="/topic/ag-sci">Agricultural Sciences</a>
                    </li>
                    <li class="topic__item">
                        <a href="/anthropology">Anthropology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/app-bio">Applied Biological Sciences</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/biochem">Biochemistry</a>
                    </li>
                    <li class="topic__item">
                        <a href="/biophysics-and-computational-biology">Biophysics and Computational Biology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/cell-bio">Cell Biology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/dev-bio">Developmental Biology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/eco">Ecology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/environmental-sciences">Environmental Sciences</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/evolution">Evolution</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/genetics">Genetics</a>
                    </li>
                </ul>
            <ul class="rlist topic__items topic__section">
                    <li class="topic__item">
                        <a href="/topic/immun">Immunology and Inflammation</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/med-sci">Medical Sciences</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/microbio">Microbiology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/neuro">Neuroscience</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/pharm">Pharmacology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/physio">Physiology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/plant-bio">Plant Biology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/pop-bio">Population Biology</a>
                    </li>
                    <li class="topic__item">
                        <a href="/psychological-and-cognitive-sciences">Psychological and Cognitive Sciences</a>
                    </li>
                    <li class="topic__item">
                        <a href="/topic/sys-bio">Systems Biology</a>
                    </li>
            </ul>
            </div>
    </div>
</div>
</div></div>

        </div>
    
</div></div><a id="menu-item-3" href="#" tabindex="" class="nav-link"></a></li><li class="nav-item px-3"><div class="covidButton"></div></li></ul></div><div class="main-menu__right-content">



        
        




        
        <a href="#" aria-expanded="false" aria-label="Expand search field" aria-controls="quick-search-form-0" data-expand-label="Expand search field" data-collapse-label="Collapse search field" data-target=".header-quick-search" data-move-focus="#header-quick-search input" data-toggle="collapse" class="btn quick-search__trigger"><svg xmlns="http://www.w3.org/2000/svg" width="45" height="45" viewBox="0 -8 32 50" class="quick-search-icon-animated"><path d="M17.71,17.91a6.25,6.25,0,1,1,1.89-4.48,6.23,6.23,0,0,1-1.89,4.48" class="cls cls-1"></path><line x1="25.07" y1="25.15" x2="8.2" y2="8.29" class="cls cls-2"></line><line x1="8.2" y1="25.15" x2="25.07" y2="8.29" class="cls cls-3"></line></svg></a>




        
        















<div class="eCommerceCartIndicatorWidget">
    
            <a href="/action/showCart?FlowID=1" id="cartLabel" title="Show shopping cart" class="shopping-cart-indicator">
                    <span class="position-relative d-flex mr-lg-3 mr-0 pt-1 pr-1 pt-lg-0 pr-lg-0">
                        <i aria-hidden="true" class="icon-shopping-cart"></i>
                        <span class="shopping-cart-indicator__badge d-none "
                              data-id="cart-size">0</span>
                    </span>
                
                <span class="shopping-cart-indicator__label font-weight-bold text-deep-gray">Cart</span>
            </a>
        
</div>




        
        




    








        




    







<div class="login-bar text-uppercase">
    <a href="/action/showLogin?data-target=.login-bar__menu&amp;data-toggle=collapse&amp;aria-expanded=true" class="font-weight-bold btn btn-outline-primary btn--light-outline btn-login collapsed" data-target=".login-bar__menu" data-toggle="collapse" aria-expanded="true">
        <span>Sign in</span>
    </a>
    
            <a href="/action/registration?redirectUri=%2Fdoi%2F10.1073%2Fpnas.0501761102" class="font-weight-bold btn btn-primary btn-register">
                Register
            </a>
        

    <div id="login-bar__menu" class="login-bar__menu collapse">
        <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1073%2Fpnas.0501761102" class="login-bar__menu__login-option">
            Individual Login
        </a>
        <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1073%2Fpnas.0501761102" class="login-bar__menu__login-option">
            Institutional Login
        </a>
    </div>
</div>
    





        
        <div class="d-flex align-items-center"><a href="https://www.pnascentral.org/cgi-bin/main.plex" class="font-weight-bold btn btn-primary btn-header-submit">Submit</a></div>
</div><!-- Institution Banner Dropzone for small devices--><div class="d-block d-xl-none">



        
        
</div></div>
</div></nav><div class="bg-black-gray header-quick-search collapse"><span class="arrow"></span><div class="quick-search--container">



        
        <div class="quick-search row no-gutters flex-column quick-search--header flex-column"><div class="quick-search__items col"><div aria-labelledby="#trigger-quick-search-form-0" id="quick-search-form-0" data-parent=".quick-search__items" role="tabpanel" class="collapse show "><form action="/action/doSearch" name="defaultQuickSearch" method="get" id="header-quick-search"><fieldset><legend class="sr-only">Quick Search anywhere</legend><div class="input-group option-0   animation-underline"><label for="AllFieldba20dcbf-b159-41b6-a89e-49552f0ecc620" class="sr-only">Enter text / DOI / keywords / authors / etc</label><input type="search" id="AllFieldba20dcbf-b159-41b6-a89e-49552f0ecc620" name="AllField" autocomplete="off" placeholder="Enter text / DOI / keywords / authors / etc" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="quick-search__input form-control autocomplete"/><div class="input-group-append"></div></div></fieldset></form></div><div aria-labelledby="#trigger-quick-search-form-1" id="quick-search-form-1" data-parent=".quick-search__items" role="tabpanel" class="collapse "><form action="/action/quickLink" name="citationQuickSearch" method="get"><fieldset class="row no-gutters"><legend class="sr-only">Quick search in Citations</legend><div class="input-group option-citation option-1"><label for="quickLinkJournalba20dcbf-b159-41b6-a89e-49552f0ecc621" class="sr-only">Journal</label><input type="search" id="quickLinkJournalba20dcbf-b159-41b6-a89e-49552f0ecc621" name="quickLinkJournal" placeholder="Journal" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-publication-titles-conf="3" value="" class="form-control autocomplete quick-search__input"/><label for="quickLinkYearba20dcbf-b159-41b6-a89e-49552f0ecc621" class="sr-only">Year</label><input type="search" id="quickLinkYearba20dcbf-b159-41b6-a89e-49552f0ecc621" name="quickLinkYear" autoComplete="false" placeholder="Year" class="form-control"/><label for="quickLinkVolumeba20dcbf-b159-41b6-a89e-49552f0ecc621" class="sr-only">Volume</label><input type="search" id="quickLinkVolumeba20dcbf-b159-41b6-a89e-49552f0ecc621" name="quickLinkVolume" autoComplete="false" placeholder="Volume" class="form-control"/><label for="quickLinkIssueba20dcbf-b159-41b6-a89e-49552f0ecc621" class="sr-only">Issue</label><input type="search" id="quickLinkIssueba20dcbf-b159-41b6-a89e-49552f0ecc621" name="quickLinkIssue" autoComplete="false" placeholder="Issue" class="form-control"/><label for="quickLinkPageba20dcbf-b159-41b6-a89e-49552f0ecc621" class="sr-only">Page</label><input type="search" id="quickLinkPageba20dcbf-b159-41b6-a89e-49552f0ecc621" name="quickLinkPage" autoComplete="false" placeholder="Page" class="form-control"/><input type="hidden" name="quickLink" value="true"/><div class="input-group-append"></div></div></fieldset></form></div><div aria-labelledby="#trigger-quick-search-form-2" id="quick-search-form-2" data-parent=".quick-search__items" role="tabpanel" class="collapse "><form action="/action/doSearch" name="publicationQuickSearch" method="get" id="header-quick-search"><fieldset><legend class="sr-only">Quick Search in Journals</legend><div class="input-group option-2 option-journal  animation-underline"><label for="AllFieldba20dcbf-b159-41b6-a89e-49552f0ecc622" class="sr-only">Enter text / DOI / keywords / authors / etc</label><input type="search" id="AllFieldba20dcbf-b159-41b6-a89e-49552f0ecc622" name="AllField" autocomplete="off" placeholder="Enter text / DOI / keywords / authors / etc" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="quick-search__input form-control autocomplete"/><input type="hidden" name="SeriesKey" value="nexus"/><div class="input-group-append"></div></div></fieldset></form></div><div aria-labelledby="#trigger-quick-search-form-3" id="quick-search-form-3" data-parent=".quick-search__items" role="tabpanel" class="collapse "><form action="/action/doSearch" name="publicationQuickSearch" method="get" id="header-quick-search"><fieldset><legend class="sr-only">Quick Search in Journals</legend><div class="input-group option-3 option-journal  animation-underline"><label for="AllFieldba20dcbf-b159-41b6-a89e-49552f0ecc623" class="sr-only">Enter text / DOI / keywords / authors / etc</label><input type="search" id="AllFieldba20dcbf-b159-41b6-a89e-49552f0ecc623" name="AllField" autocomplete="off" placeholder="Enter text / DOI / keywords / authors / etc" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="quick-search__input form-control autocomplete"/><input type="hidden" name="SeriesKey" value="pnas"/><div class="input-group-append"></div></div></fieldset></form></div></div><div class="row no-gutters justify-content-end align-items-center quick-search__actions flex-column flex-sm-row"><div class="col d-flex align-items-center flex-grow-1"><label class="mb-0 text-uppercase dropdown-selectable__label mr-2">Searching:</label><div class="d-flex align-items-center flex-fill "><div class="quick-search__dropdown dropdown dropdown-selectable"><button id="dropdownMenuButton" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn dropdown-toggle">Anywhere</button><div aria-labelledby="dropdownMenuButton" role="tablist" class="dropdown-menu"><a href="#quick-search-form-0" id="trigger-quick-search-form-0" data-toggle="collapse" class="dropdown-item  active">Anywhere</a><a href="#quick-search-form-1" id="trigger-quick-search-form-1" data-toggle="collapse" class="dropdown-item">Citation</a><a href="#quick-search-form-2" id="trigger-quick-search-form-2" data-toggle="collapse" class="dropdown-item">PNAS Nexus</a><a href="#quick-search-form-3" id="trigger-quick-search-form-3" data-toggle="collapse" class="dropdown-item  active">Proceedings of the National Academy of Sciences</a></div></div></div></div><div class="col justify-content-end d-flex align-items-center quick-search__buttons">



        
        <a href="/search/advanced" class="text-white text-uppercase mini-title quick-search__advanced">Advanced Search</a>
<button type="submit" form="header-quick-search" title="Search" class="quick-search__btn btn d-flex btn-primary align-items-center justify-content-center justify-content-sm-between mb-3 mb-sm-0"><span class="font-weight-bold text-uppercase mini-title">Search</span><i class="icon-arrow-right align-middle ml-2"></i></button></div></div></div>




        
        <div class="suggested-searches">



        
        <div class="suggested-searches__title">SUGGESTED SEARCHES: </div>
<div class="suggested-searches__search-words">
    <a href="/action/doSearch?AllField=covid+19" class="suggested-searches__search-word">Covid-19</a>
    <a href="/action/doSearch?AllField=demography" class="suggested-searches__search-word">Demography</a>
    <a href="/action/doSearch?AllField=sustainability" class="suggested-searches__search-word">Sustainability</a>
    <a href="/action/doSearch?AllField=climate+change" class="suggested-searches__search-word">Climate Change</a>
    <a href="/action/doSearch?AllField=microbiome" class="suggested-searches__search-word">Microbiome</a>
</div>
</div>
</div></div><section aria-label="slider-menu" id="slider-side-menu" class="slider"><div class="slider__container"><div id="screen-side-navigation" class="slider__screen slider__screen--root"><div class="slider__screen__content"><div class="side-menu px-3"><div class="side-menu__user-login-bar"></div><div class="side-menu__quick-search">



        
        <div class="quick-search row no-gutters"><div class="col quick-search__sidebar"><form action="/action/doSearch" name="defaultQuickSearch" method="get" id="quick-search-from-85eb734f-3d8c-4802-b914-d40de047cf3c0" role="search"><fieldset><legend class="sr-only">Quick Search anywhere</legend><div class="input-group option-0  align-items-center  animation-underline animation-underline--primary"><label for="AllField85eb734f-3d8c-4802-b914-d40de047cf3c0" class="sr-only">Enter words / phrases / DOI / ISBN / keywords / authors / etc</label><input type="search" id="AllField85eb734f-3d8c-4802-b914-d40de047cf3c0" name="AllField" autocomplete="off" placeholder="Enter words / phrases / DOI / ISBN / keywords / authors / etc" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" class="quick-search__input form-control autocomplete"/><div class="input-group-append"><button type="submit" title="Search" class="btn quick-search__btn p-0"><i class="icon-search align-middle"></i></button></div></div></fieldset></form></div></div>
</div><div class="side-menu__nav-menu"></div><div class="side-menu__extra-nav"></div><div class="side-menu__menu-footer">









        <footer data-widget-def="UX3HTMLWidget" data-widget-id="c1d89ee2-d7e8-41ac-9c96-6105d574b1d8">
        



        
          <a href="/" class="footer-logo">
    <img src="/pb-assets/images/Logos/header-logo/logo-1624644560537.svg" alt="PNAS">
  </a>
  <div class="footer__social-media__wrapper d-flex mt-3">
    <div class="footer__social-media">
      <ul role="list" class="list-inline footer__social-media__list text-center d-flex flex-wrap">
        <li role="listitem" class="list-inline-item footer__social-media__list-item">
          <a href="https://www.facebook.com/pages/PNAS/18262365099" title="Facebook" class="d-block">
            <i aria-hidden="true" class="icon-facebook"></i>
          </a>
        </li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item">
          <a href="https://twitter.com/PNASNews" title="Twitter" class="d-block">
            <i aria-hidden="true" class="icon-twitter"></i>
          </a>
        </li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item">
          <a href="https://www.youtube.com/channel/UCYjvFEQMgH6ou8fSYfiOKCw" title="Youtube" class="d-block">
            <i aria-hidden="true" class="icon-youtube"></i>
          </a>
        </li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item">
          <a href="/action/showPreferences?menuTab=Alerts" title="Mail" class="d-block">
            <i aria-hidden="true" class="icon-mail"></i>
          </a>
        </li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item">
          <a href="/about/rss" title="RSS feeds" class="d-block">
            <i aria-hidden="true" class="icon-rss"></i>
          </a>
        </li>
      </ul>
    </div>
  </div>
  <div class="footer__links">
    <a href="#">Terms of Service</a>
    <a href="#">Privacy Policy</a>
    <a href="#">Accessibility</a>
  </div>

        </footer>
    
</div></div></div></div></div></section></header>




        
        <main>



        
        











        <div data-widget-def="ux3-layout-widget" data-widget-id="d70724a7-c39c-436d-9fd8-8229e59c92e7" class="article-container bg-white mt-5">
        



        
        <div>



        
        <div class="article-container mt-5">



        
        <link id="build-style-article" rel="stylesheet" type="text/css" href="/products/pnas/releasedAssets/css/build-article-e41ad7f57abf398b554b.css"/><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-type="research-article" vocab="http://schema.org/" typeof="ScholarlyArticle"><section id="frontmatter" data-extent="frontmatter"><header><div class="core-container"><div data-article-access="free" data-article-access-type="free" class="meta-panel"><div class="meta-panel__left-content"><div class="meta-panel__type"><span>Research Article</span></div><div class="meta-panel__sub-type"><a href="/topic/bio-sci">Biological Sciences</a></div><div class="meta-panel__access meta-panel__access--free">FREE ACCESS</div></div><div class="meta-panel__right-content"><div class="meta-panel__share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share__block share__inline-links"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="d-flex list-unstyled addthis addthis_toolbox mb-0"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i></a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i></a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-LinkedIn2"></i></a></li><li><a role="link" class="addthis_button_mailto"><i aria-hidden="true" class="at-icon-wrapper icon-mail"></i></a></li></ul></div>
</div><div class="meta-panel__crossmark"><span class="crossmark"><a data-target="crossmark" href="#" data-doi="10.1073/pnas.0501761102" class="crossmark__link"><img alt="Check for updates on crossmark" src="/specs/products/pnas/releasedAssets/images/crossmark.png" class="crossmark__logo"/></a></span></div></div></div><h1 property="name">Functional genomic analysis of the rates of protein evolution</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><a href="#con1"><span property="givenName">Dennis P.</span> <span property="familyName">Wall</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con2"><span property="givenName">Aaron E.</span> <span property="familyName">Hirsh</span></a></span>, <span property="author" typeof="Person" role="listitem"><a href="#con3"><span property="givenName">Hunter B.</span> <span property="familyName">Fraser</span></a></span><span data-control-for="ui-revealable" aria-hidden="true">, <span data-action-for="ui-revealable" data-action="show">+3</span> </span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con4"><span property="givenName">Jochen</span> <span property="familyName">Kumm</span></a></span></span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con5"><span property="givenName">Guri</span> <span property="familyName">Giaever</span></a></span></span><span class="ui-revealable ui-hidden">, <span property="author" typeof="Person" role="listitem"><a href="#con6"><span property="givenName">Michael B.</span> <span property="familyName">Eisen</span></a></span></span>, and <span property="author" typeof="Person" role="listitem"><a href="#con7"><span property="givenName">Marcus W.</span> <span property="familyName">Feldman</span></a></span><span class="ui-hidden" data-control-for="ui-revealable" aria-hidden="true"> <span data-action-for="ui-revealable" data-action="hide">-3</span></span></span></span></div><div class="self-citation"><span property="datePublished">March 30, 2005</span> | <span class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume"><span property="volumeNumber">102</span></span> (<span property="isPartOf" typeof="PublicationIssue"><span property="issueNumber">15</span></span>) <span property="pagination"><span property="pageStart">5483</span>-<span property="pageEnd">5488</span></span></span> | <a href="https://doi.org/10.1073/pnas.0501761102" property="sameAs">https://doi.org/10.1073/pnas.0501761102</a></div><div data-core-tabs="relations" data-core="tabs" class="core-relations my-3"><button aria-controls="articleRelationGroup-0" class="btn--article-relation px-3 py-14 mb-1 mr-1"><span>This article has a correction</span><i aria-hidden="true" class="ml-2 icon-minus active"></i><i aria-hidden="true" class="ml-2 icon-plus"></i></button><div id="articleRelationGroup-0" tabIndex="-1" class="relationsGroup"><div class="relations-item-box mb-3 p-3"><div class="rel-heading mb-1"><div class="pill article-type">Correction</div><div class="pill article-date">May 16, 2006</div><div class="pill article-oa">FREE ACCESS</div></div><a href="/doi/10.1073/pnas.0602783103" class="rel-title">Correction for Wall et al., Functional genomic analysis of the rates of protein evolution</a><div class="rel-authors"></div></div></div></div><div class="info-panel"><div class="info-panel__metrics info-panel__item">



        
        







        <div class="toolbar-metric-container data-source" data-source="/pb/widgets/toolBarMetric/getResponse?widgetId=e3cebcd6-6709-4826-9957-d822f658d986&pbContext=%3Bjournal%3Ajournal%3Apnas%3Bpage%3Astring%3AArticle%2FChapter+View%3BrequestedJournal%3Ajournal%3Apnas%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apnas-site%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1073%2Fpnas.0501761102%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3Adoi%5C%3A10.1073%2Fpnas.2005.102.issue-15&doi=10.1073%2Fpnas.0501761102"></div>
    

</div><div class="info-panel__right-items-wrapper"><div class="info-panel__favorite info-panel__item">



        
        <div data-permission="" class="article-tools"><a href="/action/addCitationAlert?doi=10.1073%2Fpnas.0501761102" target="_blank" title="Track Citations" aria-label="Track Citations" data-toggle="tooltip" class="article-tools__citation btn"><i class="icon-bell"></i></a><a href="/personalize/addFavoritePublication?doi=10.1073%2Fpnas.0501761102" target="_blank" title="Add to Reading List" aria-label="Add to Reading List" data-toggle="tooltip" class="article-tools__favorite btn"><i class="icon-bookmark"></i></a></div>
</div><div class="info-panel__citations info-panel__item"><a href="#tab-citations" title="cite" aria-label="citation" data-toggle="tooltip" class="btn"><i class="icon-citations"></i></a></div><div class="info-panel__formats info-panel__item"><a href="https://www.pnas.org/doi/epdf/10.1073/pnas.0501761102" aria-label="PDF" class="btn-circle btn-square btn-pdf ml-2"><i class="icon-pdf-file-1"></i></a></div></div></div></div></header><nav data-core-nav="article"><header><div class="core-self-citation"><span class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Vol. <span property="volumeNumber">102</span></span> | <span property="isPartOf" typeof="PublicationIssue">No. <span property="issueNumber">15</span></span></span></div></header><ul><li><a href="#abstract">Abstract</a></li><li><a href="#sec-1">Materials and Methods</a></li><li><a href="#sec-2">Results and Discussion</a></li><li><a href="#sec-3">Concluding Remarks</a></li><li><a href="#sec-4">Supplementary Material</a></li><li><a href="#supplementary-materials">Supporting Information</a></li><li><a href="#footnotes">Notes</a></li><li><a href="#bibliography">References</a></li></ul></nav><div id="abstracts"><div class="core-container"><section id="abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Abstract</h2><div role="paragraph">The evolutionary rates of proteins vary over several orders of magnitude. Recent work suggests that analysis of large data sets of evolutionary rates in conjunction with the results from high-throughput functional genomic experiments can identify the factors that cause proteins to evolve at such dramatically different rates. To this end, we estimated the evolutionary rates of &gt;3,000 proteins in four species of the yeast genus <i>Saccharomyces</i> and investigated their relationship with levels of expression and protein dispensability. Each protein's dispensability was estimated by the growth rate of mutants deficient for the protein. Our analyses of these improved evolutionary and functional genomic data sets yield three main results. First, dispensability and expression have independent, significant effects on the rate of protein evolution. Second, measurements of expression levels in the laboratory can be used to filter data sets of dispensability estimates, removing variates that are unlikely to reflect real biological effects. Third, structural equation models show that although we may reasonably infer that dispensability and expression have significant effects on protein evolutionary rate, we cannot yet accurately estimate the relative strengths of these effects.</div></section>



        
        
   <div class="signup-alert-ad">
    <div class="d-flex justify-content-between align-items-center">
        <div>
            <h3 class="signup-alert-ad__title">Sign up for PNAS alerts.</h3>
            <p class="signup-alert-ad__content mt-2">Get alerts for new articles, or get an alert when an article is cited.</p>
        </div>
        <div> 
            <a href="/action/showPreferences?menuTab=Alerts" class="btn btn-primary mt-2 signup-alert-ad__action">Manage alerts</a>
        </div>
    </div>
    </div>
</div></div></section><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><div role="paragraph">Soon after Kimura (<a href="#ref1" role="doc-biblioref" data-xml-rid="ref1">1</a>), Ohta (<a href="#ref2" role="doc-biblioref" data-xml-rid="ref2">2</a>), and King and Jukes (<a href="#ref3" role="doc-biblioref" data-xml-rid="ref3">3</a>) proposed that much evolutionary change at the molecular level may be caused by drift and fixation of mutations that have little impact on the organism, a number of authors (<a href="#ref4" role="doc-biblioref" data-xml-rid="ref4">4</a>, <a href="#ref5" role="doc-biblioref" data-xml-rid="ref5">5</a>) offered a prediction that seemed to follow fairly directly from this view of molecular evolution. They reasoned that the strength of selection against a deleterious mutation must depend, at least in part, on the dispensability of the entire protein to the organism. Specifically, in proteins that make a smaller contribution to organismal fitness, a larger fraction of mutations would fall within the range that could be considered nearly neutral. (In this range, the product of effective population size and selection coefficient is at most 1, and the dynamics of allele frequencies are largely controlled by stochastic sampling effects.) Therefore, if protein evolution was caused in part by the drift of nearly neutral mutations, then the rate of evolution should be higher in proteins that are less important to the organism.</div><div role="paragraph">This prediction has been difficult to test because the variables are difficult to measure. As a proxy for estimates of protein dispensability, Hirsh and Fraser (<a href="#ref6" role="doc-biblioref" data-xml-rid="ref6">6</a>) used the growth rates of yeast strains in which individual genes were deleted. Although the laboratory conditions under which these strains were grown probably differed considerably from the environment relevant to the organism's evolutionary history, Hirsh and Fraser suggested that the growth rate of a deletion mutant in the laboratory might at least correlate with protein dispensability in the wild, such that estimates from a large number of proteins would reveal a statistically significant trend. To estimate the rate of evolution of a large number of proteins, they required a fully sequenced genome for comparison with yeast and therefore resorted to the evolutionarily distant nematode <i>Caenorhabditis elegans</i>. They observed a weak but highly significant correlation between their estimators of dispensability and evolutionary rate, corroborating the early prediction of nearly neutral theory. However, when proteins were separated into two categories, those that were deemed “essential,” meaning that the gene deletion effect is lethal, and those that were deemed “nonessential,” meaning that the gene deletion effect is not lethal, they did not observe a significant difference in evolutionary rate between categories. They attributed this finding to the functional form of the relationship between dispensability and evolutionary rate, suggesting that a protein whose deletion causes a substantial growth defect is, in evolutionary terms, no more dispensable than a protein that is essential for viability.</div><div role="paragraph">Several studies have offered important extensions or revisions of these findings. Jordan <i>et al.</i> (<a href="#ref7" role="doc-biblioref" data-xml-rid="ref7">7</a>) suggested that the unexpected absence of a significant difference in evolutionary rate between essential and nonessential categories may have been a result of Hirsh and Fraser's relatively small sample size (<i>n</i> = 287) and distant evolutionary comparisons. To obtain closer comparisons and a larger sample, they analyzed <i>Escherichia coli</i> proteins, and the predicted difference between essential and nonessential proteins was in fact observed. Yang <i>et al.</i> (<a href="#ref8" role="doc-biblioref" data-xml-rid="ref8">8</a>) used yeast deletion mutant growth rates and a <i>Saccharomyces cerevisiae</i>–<i>Candida albicans</i> comparison to argue that the relationship between dispensability and evolutionary rate holds among proteins with close paralogs, but not among the yeast genome's singletons. Krylov <i>et al.</i> (<a href="#ref9" role="doc-biblioref" data-xml-rid="ref9">9</a>) offered a measure of the evolutionary conservation of a gene that showed a strong association with the lethality of the deletion mutant. They used the phylogenetic distribution of orthologous sequences among seven eukaryotes to estimate a propensity for gene loss (PGL) for each sequence. PGL may function partly as an integral of sequence evolutionary rate over time, effectively reducing statistical noise and revealing clear patterns of association with functional genomic variables.</div><div role="paragraph">Each of these studies is consistent with the view that some molecular evolutionary change is caused by drift, and that this process operates more rapidly in more dispensable proteins because the efficacy of purifying selection is reduced. A more fundamental reinterpretation of the relationship between protein dispensability and evolutionary rate was offered by Pal <i>et al.</i> (<a href="#ref10" role="doc-biblioref" data-xml-rid="ref10">10</a>). Building on their previous demonstration (<a href="#ref11" role="doc-biblioref" data-xml-rid="ref11">11</a>) that highly expressed yeast genes evolve slowly, Pal <i>et al.</i> argued that dispensable proteins evolve more rapidly only because they are weakly expressed, not because there is a direct effect of dispensability on evolutionary rate. [One possible mechanistic explanation for the association between expression level and evolutionary rate is provided by the suggestion that selection favors the use of metabolically cheap and rapidly translated amino acids in highly expressed proteins (<a href="#ref12" role="doc-biblioref" data-xml-rid="ref12">12</a>, <a href="#ref13" role="doc-biblioref" data-xml-rid="ref13">13</a>).] Such translational selection would subject more highly expressed proteins to a set of constraints that are less important for weakly expressed proteins.) Using the set of yeast growth rates analyzed by Fraser <i>et al.</i> (<a href="#ref14" role="doc-biblioref" data-xml-rid="ref14">14</a>), but somewhat closer evolutionary comparisons of <i>S. cerevisiae</i> with <i>Saccharomyces pombe</i> and <i>C. albicans</i>, Pal <i>et al.</i> (<a href="#ref10" role="doc-biblioref" data-xml-rid="ref10">10</a>) showed that when they statistically controlled for expression level, the effect of dispensability on evolutionary rate completely disappeared. Rocha and Danchin (<a href="#ref15" role="doc-biblioref" data-xml-rid="ref15">15</a>) reached the same conclusion when they used functional genomic and evolutionary data from <i>E. coli</i> in a multiple regression. They showed that when the effects of expression and functional category were regressed out first, very little residual variance was explained by each gene's designation as essential or nonessential. Although their procedure is likely to reduce the apparent effect of dispensability by first removing the effect of functional category, a strong correlate of a gene's essential/nonessential designation, they concluded that protein dispensability has almost no impact on protein evolutionary rate.</div><div role="paragraph">In all of these studies, the difficulty of measuring the relevant variables without overwhelming noise or inaccuracy has remained an important obstacle. Arguably the most elusive quantity is the dispensability of a protein over the evolutionary time scales sampled by relatively distant comparisons. For example, in Pal <i>et al.</i>'s study (<a href="#ref10" role="doc-biblioref" data-xml-rid="ref10">10</a>), the complete genome closest to <i>S. cerevisiae</i> was <i>C. albicans</i>. However, recent high-throughput gene deletion studies of this species (<a href="#ref16" role="doc-biblioref" data-xml-rid="ref16">16</a>) have revealed that many genes that are essential in <i>S. cerevisiae</i> have nonessential orthologs in <i>C. albicans</i> (and vice versa), suggesting that estimates of dispensability from <i>S. cerevisiae</i> are only rough correlates of the actual level of dispensability over the sampled evolutionary period. Among prokaryotes, closer evolutionary comparisons are possible, but growth rates of deletion mutants have not been systematically measured, so studies must use exclusively binary fitness data.</div><div role="paragraph">In the present study, we address these obstacles to obtain accurate measurements of functional genomic and molecular evolutionary variables. Specifically, we address the problem in three ways. First, we analyze a substantially expanded and improved functional genomic database. Our estimates of evolutionary rate are based on four fully sequenced genomes in the genus <i>Saccharomyces</i>. In addition to improving the accuracy of ortholog designation and divergence estimation, the proximity of these comparisons should improve the reliability of dispensability measurements performed only in <i>S. cerevisiae</i>. In addition, the dispensability estimates we analyze are based on a larger number of growth replicates and an improved method of growth rate estimation. Second, we show that distinct functional genomic data sets can be cross-referenced to remove estimates that are unlikely to be accurate or biologically meaningful. Third, we use structural equation models to investigate what kinds of conclusions can be supported, given the accuracy with which variables are currently measured.</div><section id="sec-1" data-type="materials methods"><h2>Materials and Methods</h2><div role="paragraph"><b>Functional Genomic Data.</b> For one data set of dispensability estimates, labeled SGTC for Stanford Genome Technology Center, growth rates were measured by the array-based method described in ref. <a href="#ref17" role="doc-biblioref" data-xml-rid="ref17">17</a>. Six replicate growth experiments were conducted for each of two independently constructed pools of all viable homozygous yeast deletion strains. All 12 growth experiments were conducted in rich glucose medium. Data were collected at five time points for each replicate. Each deletion strain is typically represented by four hybridization signals, corresponding to tags on the array. If a tag failed to exhibit fluorescence intensity that was 4-fold higher than the mean array background at time 0, or if a tag was found to contain sequence errors (<a href="#ref17" role="doc-biblioref" data-xml-rid="ref17">17</a>), the tag was removed from the analysis. With a linear multiple regression model that allowed for time effects, replicate series effects, and series-time interactions, relative growth rate was estimated from changes in the logarithm of each tag's fluorescence over the time course of each replicate. Estimates were averaged across tags and across replicates to obtain a relative growth rate for each deletion strain. These data are available at <a href="http://chemogenomics.stanford.edu/supplements/01yfh/files/orfgenedata.txt">http://chemogenomics.stanford.edu/supplements/01yfh/files/orfgenedata.txt</a>.</div><div role="paragraph">As a second data set of dispensability estimates, labeled Warringer <i>et al.</i>, we used deletion mutant growth rates in basal synthetic medium (a standard glucose medium) reported in ref. <a href="#ref18" role="doc-biblioref" data-xml-rid="ref18">18</a>. A list of putatively essential genes was obtained from The <i>Saccharomyces</i> Genome Database (<a href="http://www.yeastgenome.org">www.yeastgenome.org</a>) and was added to both dispensability data sets. We assigned these genes a deletion mutant growth rate of zero, unless the dispensability data set indicated a nonzero value.</div><div role="paragraph">As measurements of expression, we used mRNA abundance (<a href="#ref19" role="doc-biblioref" data-xml-rid="ref19">19</a>) and the Codon Adaptation Index (CAI) (<a href="#ref20" role="doc-biblioref" data-xml-rid="ref20">20</a>). Total mRNA abundance reported in ref. <a href="#ref19" role="doc-biblioref" data-xml-rid="ref19">19</a> was determined by the method described in ref. <a href="#ref19" role="doc-biblioref" data-xml-rid="ref19">19</a>. CAI values were for <i>S. cerevisiae</i>, as reported (<a href="#ref20" role="doc-biblioref" data-xml-rid="ref20">20</a>). Use of other codon reference tables and average CAI values across all four <i>Saccharomyces</i> species did not alter the conclusions presented here. Alternate tables were based on the following gene sets: the top 20 most highly expressed genes in <i>S. cerevisiae</i> from ref. <a href="#ref19" role="doc-biblioref" data-xml-rid="ref19">19</a>, the top 50 most highly expressed genes in <i>S. cerevisiae</i> from ref. <a href="#ref19" role="doc-biblioref" data-xml-rid="ref19">19</a>, and <i>Schizosaccharomyces pombe</i> orthologs of 20 highly expressed genes in <i>S. cerevisiae</i>.</div><div role="paragraph"><b>Evolutionary Rate Estimation.</b> We obtained whole genomic sequence data, as well as ORF annotation and synteny-based orthology designation (as described in ref. <a href="#ref21" role="doc-biblioref" data-xml-rid="ref21">21</a>), for <i>Saccharomyces bayanus, Saccharomyces mikatae, Saccharomyces paradoxus</i>, and <i>S. cerevisiae</i> (<a href="#ref22" role="doc-biblioref" data-xml-rid="ref22">22</a>). Of 5,538 putatively orthologous sets of ORFs, 1,418 did not contain sequence from one or more of the species; these ORFs were excluded from further analyses to allow use of nonsynonymous divergence (dN) in the four species phylogeny as a measure of evolutionary rate (see below). Each of the remaining 4,120 ORF sets was aligned with <span data-style="small-caps">clustalw</span> 1.83, using amino acid sequences as a template for nucleotide sequences and reverse-complementing when necessary. Because the method we used to estimate evolutionary distances is based on a model of point substitutions, it was important to exclude frame shifts, whether they were caused by authentic indel mutations or sequencing errors. We therefore implemented the following filter. A majority-rule consensus amino acid sequence was constructed for each alignment. In the event that an individual sequence disagreed with the consensus at five consecutive sites in which the consensus was defined, this ortholog set was dropped from the data set. This filter resulted in the exclusion of 417 ORF sets. An additional 237 ORFs known to contain introns were also dropped, because some splice sites are uncertain and introns may result in distinctive evolutionary processes that could obscure the relationships we hoped to detect here.</div><div role="paragraph"><span data-style="small-caps">phylip</span>'s dnaml (<a href="#ref23" role="doc-biblioref" data-xml-rid="ref23">23</a>) was used to construct a maximum-likelihood tree for each of the remaining alignments. Ortholog sets that did not exhibit the consensus phylogeny ((<i>S. paradoxus, S. cerevisiae</i>), <i>S. mikatae</i>, and <i>S. bayanus</i>) were dropped. <span data-style="small-caps">paml</span>'s codeml was then used to estimate the dN, synonymous substitutions per synonymous site (synonymous divergence, dS), and their ratio (dN/dS). Two models of protein evolution were used: codeml model 0 allows for a single dN/dS value throughout the genealogy, whereas codeml model 1 allows for a different dN/dS value for each branch. The results presented below did not differ between the two models, so only model 0 results will be presented. dN/dS′, a measure of the rate of protein evolution that corrects dN/dS for selection on synonymous sites (<a href="#ref24" role="doc-biblioref" data-xml-rid="ref24">24</a>), was also estimated for the final collection of ortholog sets (<a href="#ref8" role="doc-biblioref" data-xml-rid="ref8">8</a>).</div><div role="paragraph"><b>Statistical Analyses.</b> Correlations and partial correlations were estimated by using nonparametric (Spearman's rank correlation coefficients) and parametric (Pearson's product-moment correlation coefficients) statistics. For calculation of Pearson's coefficients, measures of evolutionary rate, <i>k</i>, were first transformed according to the function <i>f</i>(<i>k</i>) = Log[<i>k</i> + 0.001]. [Addition of a small number was necessary because some variates of <i>k</i> are 0. The number 0.001 was chosen because it results in a smooth distribution of <i>f</i>(<i>k</i>) values without outliers and a linear relationship between <i>f</i>(<i>k</i>) and other variables.] <i>P</i> values were estimated by using the asymptotic approximation <span role="math"><span id="M1" class="core-tex">\(\begin{equation*}{\mathrm{t}}=r\sqrt{{\nu}/1-r^{2}}\end{equation*}\)</span></span>, where <i>r</i> is the observed correlation coefficient, and <i>ν</i> is the number of degrees of freedom (<i>n</i> – 2 for a correlation coefficient and <i>n</i> – 3 for a partial correlation coefficient) (<a href="#ref25" role="doc-biblioref" data-xml-rid="ref25">25</a>). Estimation of <i>P</i> values by randomization, rather than asymptotic approximation, would require a prohibitively large number of permutations of the data, as <i>P</i> values were generally «10<sup>–6</sup>. We therefore used the asymptotic approximation, but also performed 10<sup>6</sup> permutations to confirm that <i>P</i> values were indeed &lt;10<sup>–6</sup>. Randomization was performed according to the method prescribed in ref. <a href="#ref25" role="doc-biblioref" data-xml-rid="ref25">25</a>.</div></section><section id="sec-2"><h2>Results and Discussion</h2><div role="paragraph"><b>Relationship Between Functional Genomic Variables and Evolutionary Rate.</b> Comparative analysis of the genomes of <i>S. cerevisiae, S. mikatae, S. paradoxus</i>, and <i>S. bayanus</i> yielded 5,538 putatively orthologous sets of ORFs (<a href="#ref22" role="doc-biblioref" data-xml-rid="ref22">22</a>). These sets were realigned and subjected to filters to remove ORFs exhibiting frame shifts or atypical phylogenies (see <i>Materials and Methods</i>). These filters yielded a final set of 3,038 high-quality, four-taxa alignments, for which the following quantities were estimated by maximum likelihood: dN, dS, dS′ (see <i>Materials and Methods</i> and ref. <a href="#ref24" role="doc-biblioref" data-xml-rid="ref24">24</a>), and the ratios dN/dS and dN/dS′. These estimates of evolutionary rate are available in Table 4, which is published as <a href="http://www.pnas.org/cgi/content/full/0501761102/DC1">supporting information</a> on the PNAS web site.</div><div role="paragraph">To determine whether protein dispensability is correlated with evolutionary rate, independent of the level of expression, we calculated the partial correlation of deletion mutant growth rate with protein evolutionary rate, controlling for level of expression. (Partial correlation measures the association between two variables, statistically controlling for the effects of a third variable that could be related to each of the other two.) To ensure that results were robust to the methods by which dispensability, expression, and evolutionary rate were measured, we used two distinct measures of expression, two independent data sets of growth rate, and three measures of protein evolutionary rate. Abundance of mRNA and the CAI both have been used previously as measures of expression level (<a href="#ref10" role="doc-biblioref" data-xml-rid="ref10">10</a>, <a href="#ref12" role="doc-biblioref" data-xml-rid="ref12">12</a>). Codon bias is more likely to reflect the level of expression that is relevant to protein evolution, as it estimates expression over the recent evolutionary history of the gene, rather than at a single time point in the laboratory. Although a comparable measure of dispensability is not available, data sets of growth rates analyzed here do represent improvements over previous generations of dispensability measurements, both in terms of the number of replicates performed and the regression method used in analysis of raw data (see <i>Materials and Methods</i>).</div><div role="paragraph">Because different measures of the rate of protein evolution reflect slightly different processes, we analyzed three distinct quantities: dN in all four species of <i>Saccharomyces</i>, dN/dS, and dN/dS′. Comparison of dN across proteins with equal divergence times has been used in a number of studies (<a href="#ref6" role="doc-biblioref" data-xml-rid="ref6">6</a>, <a href="#ref9" role="doc-biblioref" data-xml-rid="ref9">9</a>, <a href="#ref10" role="doc-biblioref" data-xml-rid="ref10">10</a>). Because this measure does not involve synonymous sites, it can be informative in relatively distant comparisons; however, it does not control for differences in mutation rate across the genome, so rates of evolution measured in this way confound mutational and selective processes. The ratio dN/dS is commonly used to control for differences in mutation rate or divergence time, revealing the degree of constraint or positive selection on proteins. However, in many species, dS is also subject to selection, making dS an inaccurate measure of mutation rate and divergence time (<a href="#ref26" role="doc-biblioref" data-xml-rid="ref26">26</a>). Furthermore, the inverse relationship between the rate of dS and expression level might weaken the relationship between expression and dN/dS, partly obscuring the relationship between expression and protein evolutionary rate. Therefore, to correct for selection on synonymous sites, we here use a measure of dS adjusted according to the gene's level of codon bias (<a href="#ref24" role="doc-biblioref" data-xml-rid="ref24">24</a>).</div><div role="paragraph">The correlations and partial correlations estimating the relationship between dispensability and evolutionary rate, and between expression and evolutionary rate, are shown in <a href="#tbl1">Table 1</a>; additional correlations are provided in Table 5, which is published as <a href="http://www.pnas.org/cgi/content/full/0501761102/DC1">supporting information</a> on the PNAS web site. In addition to nonparametric statistics, parametric linear statistics were calculated to confirm robustness to statistical method, and because the structural equation models investigated below are based on normally distributed variables involved in linear relationships. Irrespective of our measures of dispensability, expression, and evolutionary rate, the growth rates of homozygous deletion mutants exhibit a highly significant correlation with rates of protein evolution, and this correlation remains highly significant when expression is partialed out. The correlation between our estimates of dispensability and evolutionary rate ranges from <i>r</i><sub>dk</sub> = 0.219 (<i>n</i> = 2931, <i>P</i> = 3 × 10<sup>–33</sup>, Spearman rank correlation between SGTC dispensability estimates and dN/dS) to <i>r</i><sub>dk</sub> = 0.274 (<i>n</i> = 2914, <i>P</i> = 2 × 10<sup>–51</sup>, Pearson correlation between Warringer <i>et al.</i> dispensability estimates and dN). The partial correlation between dispensability and evolutionary rate, controlling for level of expression, ranges from <i>r</i><sub>dk|x</sub> = 0.163 (<i>n</i> = 2768, <i>P</i> = 5 × 10<sup>–18</sup>, Spearman partial rank correlation between SGTC dispensability estimates and dN, controlling for mRNA abundance) to <i>r</i><sub>dk|x</sub> = 0.259 (<i>n</i> = 2754, <i>P</i> = 3 × 10<sup>–43</sup>, Pearson partial correlation between Warringer <i>et al.</i> dispensability estimates and dN or dN/dS′, controlling for mRNA abundance).</div><div class="figure-wrap"><header><div class="label">Table 1.</div></header><figure id="tbl1" class="table"><figcaption>Correlation and partial correlation coefficients estimating the association between protein dispensability (d) and evolutionary rate (k), and between expression level (x) and k</figcaption><div class="table-wrap"><table><thead><tr data-xml-align="center" data-xml-valign="top"><th data-xml-align="left"> Evolution rate </th><th> Dispensability </th><th><i>r</i><sub>dk</sub></th><th> Expression </th><th><i>r</i><sub>xk</sub></th><th><i>r</i><sub>dk|x</sub></th><th><i>x</i>k|d </th></tr></thead><tbody><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN/dS′ </td><td data-xml-align="left"> Warringer <i>et al.</i></td><td> 0.239 np </td><td data-xml-align="left"> mRNA abundance </td><td> -0.368 np </td><td> 0.183 np </td><td> -0.328 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.528 np </td><td> 0.190 np </td><td> -0.513 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN </td><td data-xml-align="left"> Warringer <i>et al.</i></td><td> 0.237 np </td><td data-xml-align="left"> mRNA abundance </td><td> -0.363 np </td><td> 0.181 np </td><td> -0.324 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.493 np </td><td> 0.189 np </td><td> -0.478 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN/dS′ </td><td data-xml-align="left"> SGTC </td><td> 0.230 np </td><td data-xml-align="left"> mRNA abundance </td><td> -0.368 np </td><td> 0.166 np </td><td> -0.330 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.528 np </td><td> 0.187 np </td><td> -0.516 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN </td><td data-xml-align="left"> SGTC </td><td> 0.227 np </td><td data-xml-align="left"> mRNA abundance </td><td> -0.363 np </td><td> 0.163 np </td><td> -0.325 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.493 np </td><td> 0.185 np </td><td> -0.479 np </td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN/dS′ </td><td data-xml-align="left"> Warringer <i>et al.</i></td><td> 0.274 </td><td data-xml-align="left"> mRNA abundance </td><td> -0.279 </td><td> 0.259 </td><td> -0.256 </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.522 </td><td> 0.241 </td><td> -0.505 </td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN </td><td data-xml-align="left"> Warringer <i>et al.</i></td><td> 0.274 </td><td data-xml-align="left"> mRNA abundance </td><td> -0.282 </td><td> 0.259 </td><td> -0.259 </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.509 </td><td> 0.241 </td><td> -0.491 </td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN/dS′ </td><td data-xml-align="left"> SGTC </td><td> 0.264 </td><td data-xml-align="left"> mRNA abundance </td><td> -0.279 </td><td> 0.252 </td><td> -0.258 </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.522 </td><td> 0.232 </td><td> -0.505 </td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"> dN </td><td data-xml-align="left"> SGTC </td><td> 0.264 </td><td data-xml-align="left"> mRNA abundance </td><td> -0.282 </td><td> 0.251 </td><td> -0.262 </td></tr><tr data-xml-align="center" data-xml-valign="top"><td> </td><td> </td><td> </td><td data-xml-align="left"> CAI </td><td> -0.509 </td><td> 0.232 </td><td> -0.491 </td></tr></tbody></table></div><div class="notes"><div role="doc-footnote"><i>r</i><sub>AB</sub> denotes the correlation coefficient between any two variables A and B, while <i>r</i><sub>AB|C</sub> denotes the partial correlation coefficient between any two variables, while controlling for a third, C. For description of dispensability data sets (SGTC and Warringer <i>et al.</i>), expression data sets (mRNA abundance and CAI), and evolutionary rate estimates (dN, dN/dS, and dN/dS′) see <i>Materials and Methods</i>. See <i>Results</i> for discussion of statistical significance. np denotes nonparametric correlation, all other correlation coefficients are parametric.</div></div></figure></div><div role="paragraph">As one would expect in view of the relationship between expression and dS, the ratio dN/dS shows a weaker correlation with expression level than do the other measures of evolutionary rate. Excluding dN/dS from consideration, the correlation between expression and evolutionary rate is generally highly significant and larger in magnitude than the correlation between dispensability and evolutionary rate. The ratio <i>r</i><sub>xk</sub>/<i>r</i><sub>dk</sub> ranges in magnitude from 1.02 to 1.60 (<a href="#tbl1">Table 1</a>). The ratio <i>r</i><sub>xk|d</sub>/<i>r</i><sub>dk|x</sub> ranges in magnitude from 1.00 to 2.59 (<a href="#tbl1">Table 1</a>). Whether this ratio can be interpreted as an indicator of the relative importance of different determinants of protein evolutionary rate, or of different processes in protein evolution, is discussed below.</div><div role="paragraph">Two further questions recently raised in the literature can be addressed with the data analyzed here. Observing a significant difference in evolutionary rate between essential and nonessential categories of prokaryotic proteins, Jordan <i>et al.</i> (<a href="#ref7" role="doc-biblioref" data-xml-rid="ref7">7</a>) suggested the absence of such a difference (<a href="#ref6" role="doc-biblioref" data-xml-rid="ref6">6</a>) was caused by their relatively small sample size and distant evolutionary comparison. This suggestion is borne out by the data analyzed here, as we do now find that nonessential proteins evolve significantly faster than essential ones. (Mean dN of 612 essential proteins = 0.122; mean dN of 2,285 nonessential proteins = 0.179; Mann–Whitney <i>U</i> test, <i>P</i> = 8 × 10<sup>–33</sup>. This result does not depend substantially on the measure of evolutionary rate used.) Yang <i>et al.</i> (<a href="#ref8" role="doc-biblioref" data-xml-rid="ref8">8</a>) argued that only proteins that have close paralogs exhibit a significant relationship between dispensability and evolutionary rate. As shown in <a href="#tbl2">Table 2</a>, this argument is not borne out by the data analyzed here: When we separate genes into three categories according to Yang <i>et al.</i>'s criteria (those with close paralogs, those without paralogs, and all genes) we find a significant correlation between dispensability and evolutionary rate in all three categories. The correlation, however, does appear to be particularly strong among genes with close paralogs.</div><div class="figure-wrap"><header><div class="label">Table 2.</div></header><figure id="tbl2" class="table"><figcaption><b>Pearson's correlation (<i>r</i>) between dispensability (<i>d</i>, from ref.</b> <a href="#ref18" role="doc-biblioref" data-xml-rid="ref18"><b>18</b></a><b>) and evolutionary rate (<i>k</i>, estimated by dN) among all genes and genes with (duplicates) or without (singletons) close paralogs (as defined in ref.</b> <a href="#ref8" role="doc-biblioref" data-xml-rid="ref8"><b>8</b></a><b>), and with or without controlling for expression (<i>x</i>, estimated by CAI)</b></figcaption><div class="table-wrap"><table><thead><tr data-xml-align="center" data-xml-valign="bottom"><td> </td><th colspan="2"><span> All genes <i>n</i> = 2,914 </span></th><th colspan="2"><span> Singletons <i>n</i> = 1,298 </span></th><th colspan="2"><span> Duplicates <i>n</i> = 691 </span></th></tr><tr data-xml-align="center" data-xml-valign="top"><th data-xml-align="left"><i>r</i></th><th><i>r</i></th><th><i>P</i></th><th><i>r</i></th><th><i>P</i></th><th><i>r</i></th><th><i>P</i></th></tr></thead><tbody><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"><i>r</i><sub>dk</sub></td><td> 0.274 </td><td> 2 × 10<sup>-51</sup></td><td> 0.252 </td><td data-xml-align="left"> 4 × 10<sup>-20</sup></td><td> 0.397 </td><td> 2 × 10<sup>-27</sup></td></tr><tr data-xml-align="center" data-xml-valign="top"><td data-xml-align="left"><i>r</i><sub>dk|x</sub></td><td> 0.241 </td><td> 9 × 10<sup>-40</sup></td><td> 0.162 </td><td data-xml-align="left"> 4 × 10<sup>-9</sup></td><td> 0.387 </td><td> 5 × 10<sup>-26</sup></td></tr></tbody></table></div></figure></div><div role="paragraph"><b>Cross-Referencing Functional Genomic Data Sets.</b> An important source of inaccuracy in our estimates of dispensability is that many proteins are likely to perform functions that are important in the environment relevant to yeast evolution, but superfluous in the laboratory conditions in which growth rates are measured. If such proteins are under regulatory control that allows induction under conditions in which they are useful, measurements of the dispensability of proteins that are weakly expressed in the laboratory would be more likely to reflect only experimental noise, whereas measurements of the dispensability of proteins that are strongly expressed in the laboratory would be more likely to reflect a real effect of removing a biological function.</div><div role="paragraph">An example may serve to clarify this point. Knocking out the genes specifically responsible for galactose utilization (the GAL genes) has little effect in the glucose medium in which fitness effects were measured. However, for cells growing in galactose, these genes are highly expressed and indispensable for maximal growth. If yeasts have had to metabolize galactose fairly often in their evolutionary history, then these genes would be expected to show a relatively slow rate of evolution. Thus their dispensabilities in glucose medium, where they are not expressed, are not indicative of their importance to yeast. (We note that this argument requires only that deletion of genes that are not expressed in a certain growth environment has no effect on fitness in that environment; it does not require that most genes up-regulated in a certain condition be required for growth in that condition.)</div><div role="paragraph">This consideration of the relationship between dispensability and expression in a given environment raises the prospect of using laboratory measurements of expression to improve the quality of dispensability data. Specifically, the accuracy of dispensability data would be expected to increase with the level of expression in the laboratory. We can test this prediction in two ways. First, we divide genes into quartiles according to microarray measurements of mRNA abundance (<a href="#ref19" role="doc-biblioref" data-xml-rid="ref19">19</a>). For each quartile, we calculate the correlation between our two independent data sets of dispensability. (We exclude genes required for laboratory viability, as the list of essential genes between the two data sets is virtually identical.) As expected, we find that the correlation between independent measurements of knockout mutant growth rates increases as a function of expression quartile, suggesting that the signal-to-noise ratio of dispensability data does indeed improve among higher quartiles of expression in the laboratory (Fig. 3, which is published as <a href="http://www.pnas.org/cgi/content/full/0501761102/DC1">supporting information</a> on the PNAS web site). Second, using the same division of genes into quartiles, we plot the correlation coefficient between dispensability and evolutionary rate, as a function of expression quartile. The strength of the correlation between dispensability and evolutionary rate increases with expression quartile, again suggesting that the accuracy of dispensability estimates increases among genes that are observed to be highly expressed in the laboratory (<a href="#fig1">Fig. 1<i>a</i></a>). This analysis also serves to confirm that the relationship between dispensability and evolutionary rate is not entirely mediated by expression level. The partial correlation between dispensability and evolutionary rate, controlling for level of expression, is strongest among genes for which dispensability estimates appear to be most accurate, namely, those genes that are highly expressed in the laboratory (<a href="#fig1">Fig. 1<i>b</i></a>). For additional arguments see <i>Supporting Text</i>, which is published as <a href="http://www.pnas.org/cgi/content/full/0501761102/DC1">supporting information</a> on the PNAS web site.</div><div class="figure-wrap"><header><div class="label">Fig. 1.</div></header><figure id="fig1" class="graphic"><img src="/cms/10.1073/pnas.0501761102/asset/a19ee814-1b0f-4cff-9ffc-d6958e877f4f/assets/graphic/zpq0160579630001.jpeg" height="1800" width="1576" aria-labelledby="fig1" loading="lazy" /><figcaption>The Spearman rank correlation coefficient, <i>r</i><sub>dk</sub> (<i>a</i>), and partial correlation coefficient, <i>r</i><sub>dk|x</sub> (<i>b</i>), between protein dispensability d (Warringer <i>et al.</i> data set; see <i>Materials and Methods</i>), and evolutionary rate, k, as a function of mRNA abundance quartile. Expression quartile is shown on the <i>x</i> axis. Correlation coefficient values are indicated by stars, and <i>P</i> values are indicated by diamonds. The points are joined only for visual clarity.</figcaption></figure></div><div role="paragraph"><b>Statistical Models.</b> In a number of recent studies, the relative magnitudes of partial correlation coefficients or standard partial regression coefficients have been interpreted as indicators of the relative importance of different functional variables, or even different evolutionary processes, in determining the rates of protein evolution (<a href="#ref9" role="doc-biblioref" data-xml-rid="ref9">9</a>, <a href="#ref10" role="doc-biblioref" data-xml-rid="ref10">10</a>, <a href="#ref15" role="doc-biblioref" data-xml-rid="ref15">15</a>, <a href="#ref27" role="doc-biblioref" data-xml-rid="ref27">27</a>). However, because different functional genomic variables are almost certainly measured with very different levels of accuracy, it is important to consider the potential impact of inaccurate measurement on the relative magnitudes of statistical measures of association. To this end, it is instructive to analyze a structural equation model in which statistical variation caused by uncertain measurement is partitioned from variation caused by the stochastic biological process of sequence evolution. Such a model is diagramed in <a href="#fig2">Fig. 2</a>. We use capital letters to represent true values of protein dispensability (D), expression (X), and evolutionary rate (K). The causal relationships among these variables are shown as solid arrows in <a href="#fig2">Fig. 2</a>. Dashed arrows in <a href="#fig2">Fig. 2</a> represent the introduction of variance caused by inaccurate measurement, leading to observed values of dispensability (d), expression (x), and evolutionary rate (k). Assuming variation is normally distributed and relationships are linear, the statistical relationships shown in <a href="#fig2">Fig. 2</a> are described by the following structural equations. <div class="display-formula"><div class="equation" role="math"><div class="inner"><span id="M2" class="core-tex">\(\begin{equation*}\;r_{{\mathrm{dk}}}=m_{{\mathrm{Dd}}}p_{{\mathrm{DK}}}m_{{\mathrm{Kk}}}+m_{{\mathrm{Dd}}}r_{{\mathrm{DX}}}p_{{\mathrm{XK}}}m_{{\mathrm{Kk}}} \end{equation*}\)</span></div></div><div class="label">[1a]</div></div><div class="display-formula"><div class="equation" role="math"><div class="inner"><span id="M3" class="core-tex">\(\begin{equation*}\;r_{{\mathrm{dx}}}=m_{{\mathrm{Dd}}}r_{{\mathrm{DX}}}m_{{\mathrm{Xx}}} \end{equation*}\)</span></div></div><div class="label">[1b]</div></div><div class="display-formula"><div class="equation" role="math"><div class="inner"><span id="M4" class="core-tex">\(\begin{equation*}\;r_{{\mathrm{xk}}}=m_{{\mathrm{Xx}}}p_{{\mathrm{XK}}}m_{{\mathrm{Kk}}}+m_{{\mathrm{Xx}}}r_{{\mathrm{DX}}}p_{{\mathrm{DK}}}m_{{\mathrm{Kk}}}.\;\end{equation*}\)</span></div></div><div class="label">[1c]</div></div>The correlation coefficients <i>r</i><sub>ab</sub> (<i>a</i> ≠ <i>b</i>; <i>a, b</i> ∈ {<i>d, x, k</i>}; –1 ≤ <i>r</i><sub>ab</sub> ≤ 1) are the observed associations between variables. The coefficients <i>m</i><sub>Dd</sub>, <i>m</i><sub>Xx</sub>, and <i>m</i><sub>Kk</sub> are associations between true and observed values of dispensability, expression, and evolutionary rate. We assume (hopefully not too optimistically) that 0 ≤ <i>m</i><sub>Aa</sub> ≤ 1; that is, our observed values are positively correlated with true values. The correlation coefficient <i>r</i><sub>DX</sub> (–1 ≤ <i>r</i><sub>DX</sub> ≤ 1) is the true association (unaffected by imperfect measurement) between protein dispensability and expression level, and the path coefficients, <i>p</i><sub>XK</sub> and <i>p</i><sub>DK</sub>, measure the true effects of expression, X, and dispensability, D, on evolutionary rate, K.</div><div class="figure-wrap"><header><div class="label">Fig. 2.</div></header><figure id="fig2" class="graphic"><img src="/cms/10.1073/pnas.0501761102/asset/9b667c52-ec2b-40c8-9f2e-ec534f683ffa/assets/graphic/zpq0160579630002.jpeg" height="1319" width="1800" aria-labelledby="fig2" loading="lazy" /><figcaption>Structural equation model partitioning statistical variation caused by uncertain measurement from variation caused by the stochastic biological process of sequence evolution. Capital letters represent true values of protein dispensability (D), expression (X), and evolutionary rate (K). The causal relationships among these variables are shown as solid arrows. Dashed arrows represent the introduction of variance caused by inaccurate measurement, leading to observed values of dispensability (d), expression (x), and evolutionary rate (k). The inaccuracy of measurement may be the result of several variables, including differences between laboratory and wild conditions, variance brought on by technical errors, etc.</figcaption></figure></div><div role="paragraph">We would like to use this model to address the following question. If we assume a certain level of inaccuracy in our measurements, what are the magnitudes of the true associations among dispensability, expression, and evolutionary rate that are compatible with our observed associations among these variables? Here, we will use the Pearson correlation coefficients among variables transformed to achieve linearity, as the assumptions underlying these coefficients match those of the model. For functional data, we will use the dispensability and expression data sets that show the strongest associations with evolutionary rate. The relationship between the Warringer <i>et al.</i> data set and evolutionary rate is marginally stronger than the relationship between the SGTC data set and evolutionary rate. More importantly, CAI is twice as strongly associated with evolutionary rate as is mRNA abundance. This finding is consistent with the hypothesis that CAI reflects historical expression levels relevant to protein evolution, rather than expression levels in the laboratory.</div><div role="paragraph">Our observed associations among variables are <i>r</i><sub>dk</sub> = 0.274, <i>r</i><sub>dx</sub> =–0.086, and <i>r</i><sub>xk</sub> =–0.509. We substitute these values into Eq. <b>1</b>, and then solve for <i>p</i><sub>DK</sub>, <i>p</i><sub>XK</sub>, and <i>r</i><sub>DX</sub>. Because the accuracy of measurement is unknown, or known only very roughly, we consider a range of values for <i>m</i><sub>Dd</sub> and <i>m</i><sub>Xx</sub>, the correlations between actual and observed values of dispensability and expression. Specifically, we allow measurement accuracy to range from a lower bound approximately equal to the observed association between each variable and evolutionary rate, to an upper bound of 0.9, which is likely to exceed the accuracy of measurement for expression as well as dispensability. For simplicity, we set <i>m</i><sub>Kk</sub> = 0.8; although we do not know the accuracy of our estimates of dN, it is reasonable to assume that with high-quality alignments of four species, it is fairly high.</div><div role="paragraph">In <a href="#tbl3">Table 3</a>, the values of <i>p</i><sub>DK</sub>, <i>p</i><sub>XK</sub>, and <i>r</i><sub>DX</sub> are shown for various measurement accuracies. For the range of accuracy levels shown, <i>p</i><sub>DK</sub>/<i>p</i><sub>XK</sub>, the ratio of the true effect of dispensability to that of expression on evolutionary rate, ranges in magnitude from 0.11 (for <i>m</i><sub>Dd</sub> = 0.9 and <i>m</i><sub>Xx</sub> = 0.5) to 2.7 (for <i>m</i><sub>Dd</sub> = 0.3 and <i>m</i><sub>Xx</sub> = 0.9). Thus, when we explicitly consider a plausible range of measurement error, we see that the observed associations among dispensability, expression, and evolutionary rate, are consistent with a very wide range of true impacts of dispensability and expression on evolutionary rate. Without a better understanding of the accuracy of our estimates of dispensability and expression, attempts to estimate the relative importance of dispensability and expression in determining the rate of protein evolution are premature. At this point, the best available functional genomic and evolutionary data suggest that each of these variables has an effect on protein evolution (<a href="#tbl1">Table 1</a>), but we cannot determine their relative importance.</div><div class="figure-wrap"><header><div class="label">Table 3.</div></header><figure id="tbl3" class="table"><figcaption>True path (p) and correlation (r) coefficients among true values of dispensability (D), expression (X), and evolutionary rate (K), for a range of measurement accuracy (<i>m</i>)</figcaption><div class="table-wrap"><table><thead><tr data-xml-align="center" data-xml-valign="bottom"><td> </td><th colspan="4"><span><i>m</i><sub>Dd</sub></span></th></tr><tr data-xml-align="center" data-xml-valign="top"><th data-xml-align="left"> mXx </th><th> 0.3 </th><th> 0.5 </th><th> 0.7 </th><th> 0.9 </th></tr></thead><tbody><tr data-xml-align="left" data-xml-valign="top"><td data-xml-align="center"> 0.5 </td><td> pDK = 0.61 </td><td> pDK = 0.28 </td><td> pDK = 0.19 </td><td> pDK = 0.14 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td> </td><td> pXK = -0.92 </td><td> pXK = -1.18 </td><td> pXK = -1.23 </td><td> pXK = -1.24 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td> </td><td> rDX = -0.57 </td><td> rDX = -0.34 </td><td> rDX = -0.25 </td><td> rDX = -0.19 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td data-xml-align="center"> 0.7 </td><td> pDK = 0.92 </td><td> pDK = 0.49 </td><td> pDK = 0.34 </td><td> pDK = 0.26 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td> </td><td> pXK = -0.53 </td><td> pXK = -0.79 </td><td> pXK = -0.85 </td><td> pXK = -0.87 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td> </td><td> rDX = -0.41 </td><td> rDX = -0.25 </td><td> rDX = -0.18 </td><td> rDX = -0.14 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td data-xml-align="center"> 0.9 </td><td> pDK = 1.02 </td><td> pDK = 0.57 </td><td> pDK = 0.40 </td><td> pDK = -0.31 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td> </td><td> pXK = -0.38 </td><td> pXK = -0.60 </td><td> pXK = -0.65 </td><td> pXK = -0.67 </td></tr><tr data-xml-align="left" data-xml-valign="top"><td> </td><td> rDX = -0.32 </td><td> rDX = -0.19 </td><td> rDX = -0.14 </td><td> rDX = -0.11 </td></tr></tbody></table></div><div class="notes"><div role="doc-footnote"><i>m</i> is the association between real (uppercase) and observed (lowercase) values of the three genomic variables.</div></div></figure></div><div role="paragraph">An additional analysis of evolutionary rates, motivated by our simple model (Eq. <b>1</b>), also suggests that expression and dispensability have separate effects on evolutionary rate. In view of the importance of the unknown coefficients of measurement error, <i>m</i><sub>Dd</sub> and <i>m</i><sub>Xx</sub>, we use the model to derive a testable null hypothesis that is expected to be independent of <i>m</i><sub>Dd</sub> and <i>m</i><sub>Xx.</sub> Eq. <b>1</b> was written for dN. An analogous set of equations could be written for dS. To distinguish between the two models, we append the subscript N or S to the variables k (observed evolutionary rate) and K (true evolutionary rate). Dividing Eq. <b>1a</b> by Eq. <b>1c</b>, we obtain <div class="display-formula"><div class="equation" role="math"><div class="inner"><span id="M5" class="core-tex">\(\begin{equation*} \frac{r_{{\mathrm{dk}}_{{\mathrm{N}}}}}{r_{{\mathrm{xk}}_{{\mathrm{N}}}}}=\frac{m_{{\mathrm{Dd}}}(p_{{\mathrm{DK}}_{{\mathrm{N}}}}+p_{{\mathrm{XK}}_{{\mathrm{N}}}}r_{{\mathrm{DX}}})}{m_{{\mathrm{Xx}}}(p_{{\mathrm{XK}}_{{\mathrm{N}}}}+p_{{\mathrm{DK}}_{{\mathrm{N}}}}r_{{\mathrm{DX}}}}.\;\end{equation*}\)</span></div></div></div>If the effect of dispensability on the rate of protein evolution is entirely mediated by level of expression, as has been suggested (<a href="#ref10" role="doc-biblioref" data-xml-rid="ref10">10</a>, <a href="#ref15" role="doc-biblioref" data-xml-rid="ref15">15</a>), then <i>p</i><sub>DKN</sub>, in which case <div class="display-formula"><div class="equation" role="math"><div class="inner"><span id="M6" class="core-tex">\(\begin{equation*} \frac{r_{{\mathrm{dk}}_{{\mathrm{N}}}}}{r_{{\mathrm{xk}}_{{\mathrm{N}}}}}=\frac{m_{{\mathrm{Dd}}}r_{{\mathrm{DX}}}}{m_{{\mathrm{Xx}}}}.\;\end{equation*}\)</span></div></div></div>There are no evolutionary rate variables on the right side of the equation. Therefore, when we take precisely the same steps with the equations for dS, we find that <div class="display-formula"><div class="equation" role="math"><div class="inner"><span id="M7" class="core-tex">\(\begin{equation*} \frac{r_{{\mathrm{dk}}_{{\mathrm{N}}}}}{r_{{\mathrm{xk}}_{{\mathrm{N}}}}}=\frac{r_{{\mathrm{dk}}_{{\mathrm{S}}}}}{r_{{\mathrm{xk}}_{{\mathrm{S}}}}}.\;\end{equation*}\)</span></div></div></div>Thus, if dispensability has no independent effect on dN, then the ratio of the correlation observed between dispensability and evolutionary rate to that observed between expression and evolutionary rate should be the same for dN and dS, regardless of the values of the measurement accuracy coefficients, <i>m</i><sub>Dd</sub> and <i>m</i><sub>Xx</sub>. The data show that this is not the case. Using the growth rate data of Warringer <i>et al.</i> (<a href="#ref18" role="doc-biblioref" data-xml-rid="ref18">18</a>), codon bias as a measure of expression, and transformed values of dN and dS (see <i>Materials and Methods</i>), we find that <i>r</i><sub>dk<sub>N</sub></sub>/<i>r</i><sub>xk<sub>N</sub></sub> =–0.54, whereas <i>r</i><sub>dk<sub>S</sub></sub>/<i>r</i><sub>xk<sub>S</sub></sub> =–0.18 (<i>r</i><sub>dk<sub>N</sub></sub> = 0.274, <i>r</i><sub>xk<sub>N</sub></sub> = –0.509, <i>r</i><sub>dk<sub>S</sub></sub> = 0.13, and <i>r</i><sub>xk<sub>S</sub></sub> = –0.71. Thus, if the effect of dispensability on protein evolution was mediated entirely by expression, we would expect the association between dispensability and synonymous evolution to be much stronger than is actually observed.</div></section><section id="sec-3"><h2>Concluding Remarks</h2><div role="paragraph">This article provides three main results. First, we have shown that with the best available estimates of protein dispensability, expression, and evolutionary rate in the yeast genus <i>Saccharomyces</i>, dispensability and expression have independent, significant effects on the rate of protein evolution. Second, we have shown that measurements of expression levels in the laboratory can be used to filter data sets of dispensability estimates, removing variates that are unlikely to reflect real biological effects. This result may be useful in a variety of analyses of dispensability data, including many that are not concerned with evolutionary questions (<a href="#ref17" role="doc-biblioref" data-xml-rid="ref17">17</a>, <a href="#ref28" role="doc-biblioref" data-xml-rid="ref28">28</a>, <a href="#ref29" role="doc-biblioref" data-xml-rid="ref29">29</a>). Third, we have shown that in view of the relatively low accuracy with which functional genomic variables are currently measured, recent estimates of the relative importance of different functional variables as determinants of evolutionary rate should be treated very cautiously. Much of the data remain only rough estimates of evolutionarily relevant quantities. If a gene serves an important function in yeast's natural environment, but plays little role in rapid growth in glucose medium, measurement of a deletion mutant's growth rate will not provide an accurate estimate of the gene's dispensability, even if such measurement is highly precise.</div><div role="paragraph">A potentially important source of unexplained variance in evolutionary rate that we have not investigated here is variation in the “functional density” of proteins (<a href="#ref14" role="doc-biblioref" data-xml-rid="ref14">14</a>, <a href="#ref30" role="doc-biblioref" data-xml-rid="ref30">30</a>). An essential protein in which only 10% of sites must contain specific amino acids for proper protein function is likely to evolve faster than a protein of much smaller fitness effect in which 90% of sites are similarly constrained. It will be intriguing to investigate the relationship between the rate of evolution at individual amino acid sites and estimates of the mean fitness effect of mutations at those sites.</div><div role="paragraph">Although recent work has focused on simply characterizing the relationships between functional genomic variables and evolutionary rate, the deeper interest of these relationships lies in the possibility that they might shed light on the evolutionary process. The correlation between dispensability and evolutionary rate was offered as an early prediction of the nearly neutral theory, and its corroboration could be viewed as an indication that at least some protein evolution is caused by drift. However, although the prediction was made with purifying selection in mind, it remains a possibility that the correlation between evolutionary rate and dispensability is partly caused by positive selection. For example, an hypothesis compatible with propensity for gene loss results (<a href="#ref9" role="doc-biblioref" data-xml-rid="ref9">9</a>) is that relatively dispensable proteins are more likely to be enlisted in dramatic functional changes. Polymorphism data for a large number of genes in the yeast genome will permit estimation of the frequency of positive selection for proteins of various levels of conservation, dispensability, and expression and will thus allow us to disentangle the processes of adaptive and nearly neutral evolution.</div></section><section id="sec-4" data-type="supplementary-material"><h2>Supplementary Material</h2><div class="core-supplementary-material"><div class="core-description"><div class="heading">Supporting Information</div></div></div></section><section id="supplementary-materials" class="core-supplementary-materials"><h2>Supporting Information</h2><div class="core-supplementary-material"><div class="core-description"><div class="heading">HTML Page - 01761Table4.html</div><div>HTML Page - 01761Table4.html</div></div><div class="core-link"><ul><li><a href="/doi/suppl/10.1073/pnas.0501761102/suppl_file/01761table4.html" download="01761table4.html">Download</a></li><li>2.15 MB</li></ul></div></div><div class="core-supplementary-material"><div class="core-description"><div class="heading">HTML Page - 01761Table5.html</div><div>HTML Page - 01761Table5.html</div></div><div class="core-link"><ul><li><a href="/doi/suppl/10.1073/pnas.0501761102/suppl_file/01761table5.html" download="01761table5.html">Download</a></li><li>8.57 KB</li></ul></div></div><div class="core-supplementary-material"><div class="core-description"><div class="heading">Adobe PDF - 01761Fig3.pdf</div><div>Adobe PDF - 01761Fig3.pdf</div></div><div class="core-link"><ul><li><a href="/doi/suppl/10.1073/pnas.0501761102/suppl_file/01761fig3.pdf" download="01761fig3.pdf">Download</a></li><li>46.92 KB</li></ul></div></div></section></div></section><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="footnotes"><h2>Notes</h2><div role="doc-footnote">Author contributions: D.P.W. and A.E.H. designed research; D.P.W., A.E.H., and H.B.F. performed research; D.P.W. and A.E.H. analyzed data; J.K. and G.G. contributed new reagents/analytic tools; D.P.W. and A.E.H. wrote the paper; and M.B.E. and M.W.F. helped write and edit the manuscript.</div><div role="doc-footnote">Abbreviations: SGTC, Stanford Genome Technology Center; CAI, Codon Adaptation Index; dN, nonsynonymous divergence; dS, synonymous divergence; dS′, dS adjusted for codon bias.</div></section><section id="bibliography" role="doc-bibliography"><h2>References</h2><div class="labeled" role="doc-biblioentry"><div class="label">1</div><div id="ref1" class="citations"><div class="citation"><div class="citation-content">Kimura, M. (1968) <em>Nature</em> <b><b>217</b></b><b>,</b> 624–626.</div><div class="external-links"><div><a href="https://doi.org/10.1038/217624a0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/5637732">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nature&amp;publication_year=1968&amp;pages=624&amp;doi=10.1038%2F217624a0&amp;pmid=5637732">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">2</div><div id="ref2" class="citations"><div class="citation"><div class="citation-content">Ohta, T. (1973) <em>Nature</em> <b><b>246</b></b><b>,</b> 96–98.</div><div class="external-links"><div><a href="https://doi.org/10.1038/246096a0">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/4585855">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nature&amp;publication_year=1973&amp;pages=96&amp;doi=10.1038%2F246096a0&amp;pmid=4585855">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">3</div><div id="ref3" class="citations"><div class="citation"><div class="citation-content">King, J. L. &amp; Jukes, T. H. (1969) <em>Science</em> <b><b>164</b></b><b>,</b> 788–798.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.164.3881.788">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/5767777">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Science&amp;publication_year=1969&amp;pages=788&amp;doi=10.1126%2Fscience.164.3881.788&amp;pmid=5767777">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">4</div><div id="ref4" class="citations"><div class="citation"><div class="citation-content">Kimura, M. &amp; Ohta, T. (1974) <em>Proc. Natl. Acad. Sci. USA</em> <b><b>71</b></b><b>,</b> 2848–2852.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.71.7.2848">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/4527913">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proc.+Natl.+Acad.+Sci.+USA&amp;publication_year=1974&amp;pages=2848&amp;doi=10.1073%2Fpnas.71.7.2848&amp;pmid=4527913">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">5</div><div id="ref5" class="citations"><div class="citation"><div class="citation-content">Wilson, A. C., Carlson, S. S. &amp; White, T. J. (1977) <em>Annu. Rev. Biochem.</em> <b><b>46</b></b><b>,</b> 573–639.</div><div class="external-links"><div><a href="https://doi.org/10.1146/annurev.bi.46.070177.003041">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/409339">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Annu.+Rev.+Biochem.&amp;publication_year=1977&amp;pages=573&amp;doi=10.1146%2Fannurev.bi.46.070177.003041&amp;pmid=409339">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">6</div><div id="ref6" class="citations"><div class="citation"><div class="citation-content">Hirsh, A. E. &amp; Fraser, H. B. (2001) <em>Nature</em> <b><b>411</b></b><b>,</b> 1046–1049.</div><div class="external-links"><div><a href="https://doi.org/10.1038/35082561">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11429604">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nature&amp;publication_year=2001&amp;pages=1046&amp;doi=10.1038%2F35082561&amp;pmid=11429604">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">7</div><div id="ref7" class="citations"><div class="citation"><div class="citation-content">Jordan, I. K., Rogozin, I. B., Wolf, Y. I. &amp; Koonin, E. V. (2002) <em>Genome Res.</em> <b><b>12</b></b><b>,</b> 962–968.</div><div class="external-links"><div><a href="https://doi.org/10.1101/gr.87702">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12045149">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genome+Res.&amp;publication_year=2002&amp;pages=962&amp;doi=10.1101%2Fgr.87702&amp;pmid=12045149">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">8</div><div id="ref8" class="citations"><div class="citation"><div class="citation-content">Yang, J., Gu, Z. &amp; Li, W.-H. (2003) <em>Mol. Bol. Evol.</em> <b><b>20</b></b><b>,</b> 772–774.</div><div class="external-links"><div><a href="https://doi.org/10.1093/molbev/msg078">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12679525">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Mol.+Bol.+Evol.&amp;publication_year=2003&amp;pages=772&amp;doi=10.1093%2Fmolbev%2Fmsg078&amp;pmid=12679525">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">9</div><div id="ref9" class="citations"><div class="citation"><div class="citation-content">Krylov, D. M., Wolf, Y. I., Rogozin, I. B. &amp; Koonin, E. V. (2003) <em>Genome Res.</em> <b><b>13</b></b><b>,</b> 2229–2235.</div><div class="external-links"><div><a href="https://doi.org/10.1101/gr.1589103">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14525925">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genome+Res.&amp;publication_year=2003&amp;pages=2229&amp;doi=10.1101%2Fgr.1589103&amp;pmid=14525925">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">10</div><div id="ref10" class="citations"><div class="citation"><div class="citation-content">Pal, C., Papp, B. &amp; Hurst, L. D. (2003) <em>Nature</em> <b><b>421</b></b><b>,</b> 496–497.</div><div class="external-links"><div><a href="https://doi.org/10.1038/421496b">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12556881">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nature&amp;publication_year=2003&amp;pages=496&amp;doi=10.1038%2F421496b&amp;pmid=12556881">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">11</div><div id="ref11" class="citations"><div class="citation"><div class="citation-content">Pal, C., Papp, B. &amp; Hurst, L. D. (2001) <em>Genetics</em> <b><b>158</b></b><b>,</b> 927–931.</div><div class="external-links"><div><a href="https://doi.org/10.1093/genetics/158.2.927">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11430355">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genetics&amp;publication_year=2001&amp;pages=927&amp;doi=10.1093%2Fgenetics%2F158.2.927&amp;pmid=11430355">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">12</div><div id="ref12" class="citations"><div class="citation"><div class="citation-content">Akashi, H. (2003) <em>Genetics</em> <b><b>164</b></b><b>,</b> 1291–1303.</div><div class="external-links"><div><a href="https://doi.org/10.1093/genetics/164.4.1291">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12930740">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Genetics&amp;publication_year=2003&amp;pages=1291&amp;doi=10.1093%2Fgenetics%2F164.4.1291&amp;pmid=12930740">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">13</div><div id="ref13" class="citations"><div class="citation"><div class="citation-content">Akashi, H. &amp; Gojobori, T. (2002) <em>Proc. Natl. Acad. Sci. USA</em> <b><b>99</b></b><b>,</b> 3695–3700.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.062526999">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11904428">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proc.+Natl.+Acad.+Sci.+USA&amp;publication_year=2002&amp;pages=3695&amp;doi=10.1073%2Fpnas.062526999&amp;pmid=11904428">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">14</div><div id="ref14" class="citations"><div class="citation"><div class="citation-content">Fraser, H. B., Hirsh, A. E., Steinmetz, L. M., Scharfe, C. &amp; Feldman, M. W. (2002) <em>Science</em> <b><b>296</b></b><b>,</b> 750–752.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1068696">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11976460">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Science&amp;publication_year=2002&amp;pages=750&amp;doi=10.1126%2Fscience.1068696&amp;pmid=11976460">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">15</div><div id="ref15" class="citations"><div class="citation"><div class="citation-content">Rocha, E. P. &amp; Danchin, A. (2004) <em>Mol. Biol. Evol.</em> <b><b>21</b></b><b>,</b> 108–116.</div><div class="external-links"><div><a href="https://doi.org/10.1093/molbev/msh004">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14595100">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Mol.+Biol.+Evol.&amp;publication_year=2004&amp;pages=108&amp;doi=10.1093%2Fmolbev%2Fmsh004&amp;pmid=14595100">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">16</div><div id="ref16" class="citations"><div class="citation"><div class="citation-content">Roemer, T., Jiang, B., Davison, J., Ketela, T., Veillette, K., Breton, A., Tandia, F., Linteau, A., Sillaots, S., Marta, C., <i>et al</i>. (2003) <em>Mol. Microbiol.</em> <b><b>50</b></b><b>,</b> 167–181.</div><div class="external-links"><div><a href="https://doi.org/10.1046/j.1365-2958.2003.03697.x">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14507372">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Mol.+Microbiol.&amp;publication_year=2003&amp;pages=167&amp;doi=10.1046%2Fj.1365-2958.2003.03697.x&amp;pmid=14507372">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">17</div><div id="ref17" class="citations"><div class="citation"><div class="citation-content">Giaever, G., Flaherty, P., Kumm, J., Proctor, M., Nislow, C., Jaramillo, D. F., Chu, A. M., Jordan, M. I., Arkin, A. P. &amp; Davis, R. W. (2004) <em>Proc. Natl. Acad. Sci. USA</em> <b><b>101</b></b><b>,</b> 793–798.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.0307490100">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14718668">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proc.+Natl.+Acad.+Sci.+USA&amp;publication_year=2004&amp;pages=793&amp;doi=10.1073%2Fpnas.0307490100&amp;pmid=14718668">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">18</div><div id="ref18" class="citations"><div class="citation"><div class="citation-content">Warringer, J., Ericson, E., Fernandez, L., Nerman, O. &amp; Blomberg, A. (2003) <em>Proc. Natl. Acad. Sci. USA</em> <b><b>100</b></b><b>,</b> 15724–15729.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.2435976100">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/14676322">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proc.+Natl.+Acad.+Sci.+USA&amp;publication_year=2003&amp;pages=15724&amp;doi=10.1073%2Fpnas.2435976100&amp;pmid=14676322">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">19</div><div id="ref19" class="citations"><div class="citation"><div class="citation-content">Wang, Y., Liu, C. L., Storey, J. D., Tibshirani, R. J., Herschlag, D. &amp; Brown, P. O. (2002) <em>Proc. Natl. Acad. Sci. USA</em> <b><b>99</b></b><b>,</b> 5860–5865.</div><div class="external-links"><div><a href="https://doi.org/10.1073/pnas.092538799">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11972065">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Proc.+Natl.+Acad.+Sci.+USA&amp;publication_year=2002&amp;pages=5860&amp;doi=10.1073%2Fpnas.092538799&amp;pmid=11972065">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">20</div><div id="ref20" class="citations"><div class="citation"><div class="citation-content">Coghlan, A. &amp; Wolfe, K. H. (2000) <em>Yeast</em> <b><b>16</b></b><b>,</b> 1131–1145.</div><div class="external-links"><div><a href="https://doi.org/10.1002/1097-0061(20000915)16:12%3C1131::AID-YEA609%3E3.0.CO;2-F">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/10953085">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Yeast&amp;publication_year=2000&amp;pages=1131&amp;doi=10.1002%2F1097-0061%2820000915%2916%3A12%253C1131%3A%3AAID-YEA609%253E3.0.CO%3B2-F&amp;pmid=10953085">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">21</div><div id="ref21" class="citations"><div class="citation"><div class="citation-content">Kamvysselis, M., Patterson, N., Birren, B., Berger, B. &amp; Lander, E. (2003) in <em>Seventh Annual International Conference on Computational Molecular Biology</em>, eds. Vingron, M., Istrail, S., Pevzner, P. &amp; Waterman, M. (Assoc. Computing Machinery, Berlin), pp. 157–166.</div><div class="external-links"><div><a href="https://scholar.google.com/scholar_lookup?title=Seventh+Annual+International+Conference+on+Computational+Molecular+Biology&amp;publication_year=2003&amp;pages=157">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">22</div><div id="ref22" class="citations"><div class="citation"><div class="citation-content">Kellis, M., Patterson, N., Endrizzi, M., Birren, B. &amp; Lander, E. S. (2003) <em>Nature</em> <b><b>423</b></b><b>,</b> 241–254.</div><div class="external-links"><div><a href="https://doi.org/10.1038/nature01644">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12748633">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nature&amp;publication_year=2003&amp;pages=241&amp;doi=10.1038%2Fnature01644&amp;pmid=12748633">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">23</div><div id="ref23" class="citations"><div class="citation"><div class="citation-content">Felsenstein, J. (1989) <em>Cladistics</em> <b><b>5</b></b><b>,</b> 164–166.</div><div class="external-links"><div><a href="https://scholar.google.com/scholar_lookup?title=Cladistics&amp;publication_year=1989&amp;pages=164">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">24</div><div id="ref24" class="citations"><div class="citation"><div class="citation-content">Hirsh, A. E., Fraser, H. B. &amp; Wall, D. P. (2005) <em>Mol. Biol. Evol.</em> <b><b>22</b></b><b>,</b> 174–177.</div><div class="external-links"><div><a href="https://doi.org/10.1093/molbev/msh265">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/15371530">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Mol.+Biol.+Evol.&amp;publication_year=2005&amp;pages=174&amp;doi=10.1093%2Fmolbev%2Fmsh265&amp;pmid=15371530">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">25</div><div id="ref25" class="citations"><div class="citation"><div class="citation-content">Legendre, P. (2000) <em>J. Stat. Comput. Simul.</em> <b><b>67</b></b><b>,</b> 37–73.</div><div class="external-links"><div><a href="https://doi.org/10.1080/00949650008812035">Crossref</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=J.+Stat.+Comput.+Simul.&amp;publication_year=2000&amp;pages=37&amp;doi=10.1080%2F00949650008812035">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">26</div><div id="ref26" class="citations"><div class="citation"><div class="citation-content">Akashi, H. (2001) <em>Curr. Opin. Genet. Dev.</em> <b><b>11</b></b><b>,</b> 660–666.</div><div class="external-links"><div><a href="https://doi.org/10.1016/S0959-437X(00)00250-1">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11682310">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Curr.+Opin.+Genet.+Dev.&amp;publication_year=2001&amp;pages=660&amp;doi=10.1016%2FS0959-437X%2800%2900250-1&amp;pmid=11682310">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">27</div><div id="ref27" class="citations"><div class="citation"><div class="citation-content">Jordan, I. K., Wolf, Y. I. &amp; Koonin, E. V. (2003) <em>BMC Evol. Biol.</em> <b><b>3</b></b><b>,</b> 1.</div><div class="external-links"><div><a href="https://doi.org/10.1186/1471-2148-3-1">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12515583">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=BMC+Evol.+Biol.&amp;publication_year=2003&amp;pages=1&amp;doi=10.1186%2F1471-2148-3-1&amp;pmid=12515583">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">28</div><div id="ref28" class="citations"><div class="citation"><div class="citation-content">Steinmetz, L. M., Scharfe, C., Deutschbauer, A. M., Mokranjac, D., Herman, Z. S., Jones, T., Chu, A. M., Giaever, G., Prokisch, H., Oefner, P. J. &amp; Davis, R. W. (2002) <em>Nat. Genet.</em> <b><b>31</b></b><b>,</b> 400–404.</div><div class="external-links"><div><a href="https://doi.org/10.1038/ng929">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/12134146">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Nat.+Genet.&amp;publication_year=2002&amp;pages=400&amp;doi=10.1038%2Fng929&amp;pmid=12134146">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">29</div><div id="ref29" class="citations"><div class="citation"><div class="citation-content">Tong, A. H., Evangelista, M., Parsons, A. B., Xu, H., Bader, G. D., Page, N., Robinson, M., Raghibizadeh, S., Hogue, C. W., Bussey, H., <i>et al</i>. (2001) <em>Science</em> <b><b>294</b></b><b>,</b> 2364–2368.</div><div class="external-links"><div><a href="https://doi.org/10.1126/science.1065810">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/11743205">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=Science&amp;publication_year=2001&amp;pages=2364&amp;doi=10.1126%2Fscience.1065810&amp;pmid=11743205">Google Scholar</a></div></div></div></div></div><div class="labeled" role="doc-biblioentry"><div class="label">30</div><div id="ref30" class="citations"><div class="citation"><div class="citation-content">Zuckerkandl, E. (1976) <em>J. Mol. Evol.</em> <b><b>7</b></b><b>,</b> 167–183.</div><div class="external-links"><div><a href="https://doi.org/10.1007/BF01731487">Crossref</a></div><div><a href="https://www.ncbi.nlm.nih.gov/pubmed/933174">PubMed</a></div><div><a href="https://scholar.google.com/scholar_lookup?title=J.+Mol.+Evol.&amp;publication_year=1976&amp;pages=167&amp;doi=10.1007%2FBF01731487&amp;pmid=933174">Google Scholar</a></div></div></div></div></div></section></div></section><div class="core-collateral"><div id="core-collateral-info" role="tabpanel"><header><h2><i class="icon-info" aria-hidden="true"></i>Information &amp; Authors</h2></header><section id="tab-information" aria-labelledby="information" role="tabpanel"><h3>Information</h3><section class="core-self-citation"><h4>Published in</h4><div class="core-journal-presentation">



        
        <div class="cover-image d-flex justify-content-center flex-column"><div data-is-viewable="true" class="cover-image__image"><a title="Go to Proceedings of the National Academy of Sciences " class="d-block"><img src="/cms/asset/df45f65c-f91e-4baf-baee-8a3b2063747f/pnas.2005.102.issue-15.cover.gif" alt="Go to Proceedings of the National Academy of Sciences " loading="lazy"/></a></div><button class="cover-image__view-cover mt-3x btn btn-sm btn-outline-secondary align-self-center font-weight-500">View Cover</button><div data-cover-src="/cms/asset/df45f65c-f91e-4baf-baee-8a3b2063747f/pnas.2005.102.issue-15.cover.gif" class="cover-image__popup-moving-cover position-fixed d-block"></div><div class="cover-image__popup-view position-fixed"><div class="cover-image__popup-view__wrapper pt-lg-3 position-relative"><div class="cover-image__popup-view__close-popup position-absolute"><button class="btn text-white p-0"><i aria-hidden="true" role="button" aria-label="Close popup" class="icon-close text-lg"></i></button></div><div class="container-fluid"><div class="row"><div class="col-12 col-lg-8"><div class="cover-image__popup-view__cover"><img src="/cms/asset/df45f65c-f91e-4baf-baee-8a3b2063747f/pnas.2005.102.issue-15.cover.gif" alt="Go to Proceedings of the National Academy of Sciences " loading="lazy"/></div></div><div class="col-12 col-lg-4"><div class="cover-image__popup-view__caption-wrapper text-white"></div></div></div></div></div></div></div>
<div class="core-journal-description"><div property="isPartOf" typeof="Periodical"><span property="name">Proceedings of the National Academy of Sciences</span></div><div class="core-enumeration"><span property="isPartOf" typeof="PublicationVolume">Vol. <span property="volumeNumber">102</span></span> | <span property="isPartOf" typeof="PublicationIssue">No. <span property="issueNumber">15</span></span><br /><span property="datePublished">April 12, 2005</span></div><div class="core-pmid"><span class="heading">PubMed</span>: <a class="content" href="https://pubmed.ncbi.nlm.nih.gov/15800036/" property="sameAs" target="_blank" title="Link to PubMed">15800036</a></div></div></div></section><section class="core-classifications"><h4>Classifications</h4><div class="keywords"><ol><li><a href="/topic/bio-sci" alt="View all articles in Biological Sciences">Biological Sciences</a></li><li><a href="/topic/evolution" alt="View all articles in Evolution">Evolution</a></li><li role="presentation" aria-hidden="true" class="list-line-break"></li></ol></div></section><section class="core-copyright"><h4>Copyright</h4><div role="paragraph">Copyright © 2005, The National Academy of Sciences. Freely available online through the PNAS open access option.</div></section><section class="core-history"><h4>Submission history</h4><div><b class="core-label">Received</b>: June 11, 2004</div><div><b class="core-label">Published online</b>: March 30, 2005</div><div><b class="core-label">Published in issue</b>: April 12, 2005</div></section><section class="core-permissions"><h4>Permissions</h4><div>Request permissions for this article.</div><div><a href="/author-center/fees-and-licenses">Request Permissions</a></div></section><section property="keywords"><h4>Keywords</h4><ol><li><a href="/action/doSearch?AllField=protein+dispensability&amp;stemming=yes&amp;publication=pnas" alt="Search on this keyword">protein dispensability</a></li><li><a href="/action/doSearch?AllField=protein+fitness&amp;stemming=yes&amp;publication=pnas" alt="Search on this keyword">protein fitness</a></li><li><a href="/action/doSearch?AllField=structural+equation+models&amp;stemming=yes&amp;publication=pnas" alt="Search on this keyword">structural equation models</a></li></ol></section><section class="core-acknowledgments"><h4>Acknowledgments</h4><div role="paragraph">We thank two anonymous reviewers for their helpful comments and suggestions and Anders Blomberg and Jonas Warringer for providing fitness data. D.P.W. acknowledges support from a National Science Foundation Biological Informatics postdoctoral fellowship. A.E.H. was supported by National Institutes of Health Grant GM28428 (to M.W.F.). H.B.F. is a National Science Foundation Predoctoral Fellow.</div></section></section><section id="tab-contributors" aria-labelledby="contributors" role="tabpanel"><h3>Authors</h3><section class="core-authors"><h4>Affiliations</h4><div id="con1" property="author" typeof="Person"><div class="heading"><span property="givenName">Dennis P.</span> <span property="familyName">Wall</span><sup class="xref"><a href="#cor1" role="doc-noteref">‡</a>,<a href="#fn1" role="doc-noteref">†</a></sup></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Sciences, and Stanford Genome Technology Center, Stanford University, Stanford, CA 94305; and Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720</span></div></div><div class="core-author-link"><a href="/authored-by/Wall/Dennis+P">View all articles by this author</a></div></div></div><div id="con2" property="author" typeof="Person"><div class="heading"><span property="givenName">Aaron E.</span> <span property="familyName">Hirsh</span><sup class="xref"><a href="#fn1" role="doc-noteref">†</a></sup></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Sciences, and Stanford Genome Technology Center, Stanford University, Stanford, CA 94305; and Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720</span></div></div><div class="core-author-link"><a href="/authored-by/Hirsh/Aaron+E">View all articles by this author</a></div></div></div><div id="con3" property="author" typeof="Person"><div class="heading"><span property="givenName">Hunter B.</span> <span property="familyName">Fraser</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Sciences, and Stanford Genome Technology Center, Stanford University, Stanford, CA 94305; and Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720</span></div></div><div class="core-author-link"><a href="/authored-by/Fraser/Hunter+B">View all articles by this author</a></div></div></div><div id="con4" property="author" typeof="Person"><div class="heading"><span property="givenName">Jochen</span> <span property="familyName">Kumm</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Sciences, and Stanford Genome Technology Center, Stanford University, Stanford, CA 94305; and Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720</span></div></div><div class="core-author-link"><a href="/authored-by/Kumm/Jochen">View all articles by this author</a></div></div></div><div id="con5" property="author" typeof="Person"><div class="heading"><span property="givenName">Guri</span> <span property="familyName">Giaever</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Sciences, and Stanford Genome Technology Center, Stanford University, Stanford, CA 94305; and Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720</span></div></div><div class="core-author-link"><a href="/authored-by/Giaever/Guri">View all articles by this author</a></div></div></div><div id="con6" property="author" typeof="Person"><div class="heading"><span property="givenName">Michael B.</span> <span property="familyName">Eisen</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Sciences, and Stanford Genome Technology Center, Stanford University, Stanford, CA 94305; and Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720</span></div></div><div class="core-author-link"><a href="/authored-by/Eisen/Michael+B">View all articles by this author</a></div></div></div><div id="con7" property="author" typeof="Person"><div class="heading"><span property="givenName">Marcus W.</span> <span property="familyName">Feldman</span></div><div class="content"><div class="affiliations"><div property="affiliation" typeof="Organization"><span property="name">Department of Biological Sciences, and Stanford Genome Technology Center, Stanford University, Stanford, CA 94305; and Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720</span></div></div><div class="core-author-link"><a href="/authored-by/Feldman/Marcus+W">View all articles by this author</a></div></div></div></section><section class="core-authors-notes"><h4>Notes</h4><div class="labeled" role="doc-footnote"><div id="cor1" role="paragraph"> To whom correspondence should be sent at the present address: Department of Systems Biology, Harvard Medical School, Boston, MA 02115. E-mail: <a href="/cdn-cgi/l/email-protection#3d594d4a5c51517d55504e13555c4f4b5c4f5913585948"><span class="__cf_email__" data-cfemail="dfbbafa8beb3b39fb7b2acf1b7beada9beadbbf1babbaa">[email&#160;protected]</span></a>. </div></div><div class="labeled" role="doc-footnote"><div id="fn1" role="paragraph">D.P.W. and A.E.H. contributed equally to this work.</div></div><div role="doc-footnote">Communicated by Marc W. Kirschner, Harvard Medical School, Boston, MA, March 4, 2005</div></section></section></div><div id="core-collateral-metrics" role="tabpanel"><header><h2><i class="icon-metrics" aria-hidden="true"></i>Metrics &amp; Citations</h2></header><section id="tab-metrics-inner" aria-labelledby="metrics-inner" role="tabpanel"><h3>Metrics</h3><section>



        
        







                <div class="sidebar-metric-container data-source" data-source="/pb/widgets/fullSideBarMetric/getResponse?widgetId=8cd1bebc-5b8f-497f-a198-32e453e000d3&pbContext=%3Bjournal%3Ajournal%3Apnas%3Bpage%3Astring%3AArticle%2FChapter+View%3BrequestedJournal%3Ajournal%3Apnas%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apnas-site%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1073%2Fpnas.0501761102%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3Adoi%5C%3A10.1073%2Fpnas.2005.102.issue-15&doi=10.1073%2Fpnas.0501761102"></div>
        
</section><section>



        
            <h5 class="">
                Citation statements
            </h5>
        
        <div class="scite-badge py-3"
data-doi="10.1073/pnas.0501761102"
data-layout="horizontal"
data-tooltip-placement="bottom"
data-show-zero="true"
data-show-labels="false">
</div>
<script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script async type="application/javascript" src="https://cdn.scite.ai/badge/scite-badge-latest.min.js">
</script>




        
            <h5 class="">
                Altmetrics
            </h5>
        
        <div data-badge-details="right" data-badge-type="large-donut" data-doi="10.1073/pnas.0501761102" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed py-3">
</div>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
</section></section><section id="tab-citations" aria-labelledby="citations" role="tabpanel"><h3>Citations</h3><section></section><section>



        
        <div class="citation-download pb-5">
	<p class="citation-msg">
		If you have the appropriate software installed, you can download article citation data to the citation manager of your choice. Simply select your manager software from the list below and click Download.
	</p>
	<!-- download options -->
	<form action="/action/downloadCitation" name="frmCitmgr" class="citation-form" method="post" target="_self"><input type="hidden" name="doi" value="10.1073/pnas.0501761102"><input type="hidden" name="downloadFileName" value="csp_102_"><input type="hidden" name="include" value="abs">
		<fieldset class="format-select">
			<label for="slct_format">Format</label>
			<div class="select-container">
				<select id="slct_format" name="format" class="js__slcInclude">
					<option value="" selected="selected">Please select one from the list</option>
					<option value="ris">RIS (ProCite, Reference Manager)</option>
					<option value="endnote">EndNote</option>
					<option value="bibtex">BibTex</option>
					<option value="medlars">Medlars</option>
					<option value="refworks">RefWorks</option>
				</select>
			</div>
			<div class="form-check custom-control custom-checkbox py-2"><input id="direct-import" type="checkbox" name="direct" value="true" checked="checked" class="custom-control-input"><label for="direct-import" class="custom-control-label text-uppercase font-weight-bold">Direct Import</label></div>
                                                                                
                                                                
                 
           
		</fieldset>           
		<footer class="form-footer">
			<input onclick="onCitMgrSubmit()" class="btn btn-primary" type="submit" name="submit" value="Download">
		</footer>
	</form>
</div>




        
        <section class="pill__item"><h4>Cited by</h4><section id="cited-by" class="cited-by"><div class="cited-by__wrapper"><div id="cited-by__content" class="cited-by__content"><ol data-total='8' data-source="/pb/widgets/citedBy?page=0&amp;widgetId=b459f3f0-560a-46b6-9bf5-038940b8c01f&amp;pbContext=%3Bjournal%3Ajournal%3Apnas%3Bpage%3Astring%3AArticle%2FChapter+View%3BrequestedJournal%3Ajournal%3Apnas%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apnas-site%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1073%2Fpnas.0501761102%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3Adoi%5C%3A10.1073%2Fpnas.2005.102.issue-15" data-lazyload="true" class="cited-by__list list-unstyled"></ol></div><div class="cited-by__spinner justify-content-center align-items-center d-none"><div role="status" class="spinner-border text-gray"><span class="sr-only">Loading...</span></div></div></div></section></section>
</section></section></div><div id="core-collateral-fulltext-options" role="tabpanel"><header><h2><i class="icon-eye" aria-hidden="true"></i>View Options</h2></header><div tabindex="0" class="section--wrapper"><h3>View options</h3><section class="pill__item"><h4>PDF format</h4>



        
        <div class="py-3">Download this article as a PDF file</div>
<a href="https://www.pnas.org/doi/epdf/10.1073/pnas.0501761102" class="btn btn-pdf"><i aria-hidden="true" class="icon-pdf-file-1"></i><span>Download PDF</span></a></section><!--if(xmlFormat && ccByLicense)section.pill__item
    h4 XML format
    | 
    a(href=xmlFormat.link).btn.btn-outline-primary
        i.icon-download(aria-hidden="true")
        span Download XML--></div><!-- Its needed to duplicate this for PB check Pill.js as well--><div tabindex="-1" class="section--wrapper"><h3>Get Access</h3><section>









        <div data-widget-def="eCommercePurchaseAccessWidget" data-widget-id="f9ad4b91-1d08-4a0f-9bf7-163286863c86" class="eCommercePurchaseAccessWidget">
        



        
        











    <div class="pill__item pill__item--border-bottom purchaseArea ">
        




<div class="loginOptions">
    <h4>Login options</h4>
    <p>
        Check if you have access through your login credentials or your institution to get full access on this article.
    </p>
    
            <a href="/action/showLogin?redirectUri=%2Fdoi%2F10.1073%2Fpnas.0501761102" class="btn btn-primary btn-sm mr-1">
                <i aria-hidden="true" class="icon-user-o"></i>
                Personal login
            </a>
        
    <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1073%2Fpnas.0501761102" class="btn btn-primary btn-sm">
    <i class="icon-institution"></i>
    <span>Institutional Login</a></span>
    
</div>

<div class="recommendToLibrarian">
    <h4>Recommend to a librarian</h4>
    <a href="https://app.smartsheet.com/b/form/a5f8b19a96f343cfa680d55801fd373c" style="btn btn-primary btn-sm mr-">
        Recommend PNAS to a Librarian
    </a>
</div>
    </div>
    <div class="pill__item eCommercePurchaseAccessWidgetContainer">
        
                <div class="save-for-later">
                    
                        <a data-bind="saveItem" data-doi="10.1073/pnas.0501761102" class="save-for-later-link"
                           href="/action/saveItem?doi=10.1073/pnas.0501761102">
                            <i aria-hidden="true" class="icon-heart"></i>
                            Save for later
                        </a>
                    
                    <a class="saved-go-cart d-none" href="/action/showCart?FlowID=2">Item saved, go to cart</a>
                </div>
            
        <h4>Purchase options</h4>
        <p>Purchase this article to get full access to it.</p>
        











<div class="purchase-options-container">
    

    
        <div class="savedItem-info d-none">
            <p></p>
        </div>
    

    
            
                
                
                        <div class="tab-content">
                            <div class="purchase-items">
                                <div class="purchase-item">
                                    <div class="tab-pane-content">
                                        <div class="purchaseMessage d-none" data-item="PPV-PPV-10-ELECTRONIC-USD-10.1073/PNAS.0501761102-500047">
                                            <p></p>
                                        </div>
                                        <header>
                                            <p class="purchase-item__title">Single Article Purchase
                                                

                                                    







<div class="usageRight">
    
    
        
        

    
        
        

    
        
        

    
        
        

    
</div>
                                                
                                            </p>
                                            <div class="purchaseMessage d-none info" data-item="PPV-PPV-10-ELECTRONIC-USD-10.1073/PNAS.0501761102-500047">
                                                <p></p>
                                            </div>
                                        </header>
                                        <b>Functional genomic analysis of the rates of protein evolution</b>
                                        <div class="purchase-item__info">
                                            <div>Proceedings of the National Academy of Sciences</div>
                                            <ul class="list-list-unstyled list-inline">
                                                <li class="list-inline-item">Vol. 102</li>
                                                <li class="list-inline-item">No. 15</li>
                                                
                                                    <li class="list-inline-item">pp. 5301-5635</li>
                                                
                                            </ul>
                                        </div>
                                        <footer>
                                            <div class="purchase-item__price-group"
                                                 style="">
                                                <div class="purchase-item__price">
                                                    <span class="purchase-item__price__value">
                                                            $10.00
                                                    </span>
                                                </div>
                                                <button data-bind="addItem" data-key="PPV-PPV-10-ELECTRONIC-USD-10.1073/PNAS.0501761102-500047"
                                                        data-entity="PPV-PPV-10-ELECTRONIC-USD-10.1073/PNAS.0501761102-500047"
                                                        href="/action/addToCart?id=PPV-PPV-10-ELECTRONIC-USD-10.1073/PNAS.0501761102-500047"
                                                        class="add-article-to-cart btn btn-primary btn-sm">
                                                    <i aria-hidden="true" class="icon-shopping-cart"></i>
                                                    <span>Add to Cart</span>
                                                </button>
                                            </div>
                                            <div data-item="PPV-PPV-10-ELECTRONIC-USD-10.1073/PNAS.0501761102-500047"
                                                 class="addedMessage d-none">
                                                <a href="/action/showCart?FlowID=2" class="btn btn-success">
                                                    Checkout
                                                </a>
                                            </div>
                                        </footer>
                                    </div>
                                </div>
                            </div>
                        </div>
                    
            
        

    
</div>
    </div>




        </div>
    




        
        <div class="pill__item"><h4>Restore content access</h4><p><a href="/action/showRestoreContentAccess">Restore content access for purchases made as a guest</a></p></div>
</section></div></div><div id="core-collateral-media" role="tabpanel"><header><h2><i class="icon-figures" aria-hidden="true"></i>Media</h2></header><section id="tab-figures" aria-labelledby="figures" role="tabpanel"><h3>Figures</h3></section><section id="tab-tables" aria-labelledby="tables" role="tabpanel"><h3>Tables</h3></section><section id="tab-other" aria-labelledby="other" role="tabpanel"><h3>Other</h3></section></div><div id="core-collateral-share" role="tabpanel"><header><h2><i class="icon-share" aria-hidden="true"></i>Share</h2></header><h3>Share</h3><div class="section--wrapper"><section><h4>Share article link</h4><div id="share-self"><p class="share-self__source"></p><button data-id="article-share-access" class="share-self__action btn btn-outline-dark"><i aria-hidden="true" class="icon-copy"></i><span>Copy Link</span></button><div class="share-self__status"><p class="share-self__success"><i aria-hidden="true" class="icon-check_circle"></i><span>Copied!</span></p><p class="share-self__failed"><i aria-hidden="true" class="icon-x_btnclose"></i><span>Copying failed.</span></p></div></div></section><section><h4>Share on social media</h4><div class="share-buttons"><a role="link" target="_blank" href="#" data-id="article-share-facebook" class="btn mr-3 mb-3 btn--facebook addthis_button_facebook"><i aria-hidden="true" class="icon-facebook"></i><span>Facebook</span></a><a role="link" target="_blank" href="#" data-id="article-share-twitter" class="btn mr-3 mb-3 btn--twitter addthis_button_twitter"><i aria-hidden="true" class="icon-twitter"></i><span>Twitter</span></a><a role="link" target="_blank" href="#" data-id="article-share-linkedin" class="btn mr-3 mb-3 btn--linkedin addthis_button_linkedin"><i aria-hidden="true" class="icon-LinkedIn2"></i><span>Linkedin</span></a><a role="link" target="_blank" href="#" data-id="article-share-gmail" class="btn mr-3 mb-3 btn--gmail addthis_button_gmail"><i aria-hidden="true" class="icon-gmail"></i><span>Gmail</span></a><a role="link" target="_blank" href="#" data-id="article-share-email" class="btn mr-3 mb-3 btn--email addthis_button_email"><i aria-hidden="true" class="icon-mail"></i><span>email</span></a></div></section></div></div></div></article><div data-extent="backmatter" class="after-credits"><div class="core-container text-center"><a href="/doi/full/10.1073/pnas.0501761102">VIEW FULL TEXT</a><a href="/doi/pdf/10.1073/pnas.0501761102">DOWNLOAD PDF</a></div></div><script id="contribution-meta" type="application/json">{
    "doi": "10.1073/pnas.0501761102",
    "toc": "/toc/pnas/102/15",
    "issn": "0027-8424"
}</script><div class="articles-nav"></div>




        
        
</div>
</div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="80a46abd-a91b-4297-bdf4-439b723e0ae1" class="article-further-reading multi-search--grid py-2_5x bg-deep-gray card-dark">
        



        
        <div class="container-fluid">









        <div data-widget-def="ux3-layout-widget" data-widget-id="361aa181-1ee7-40d6-9058-915ead64903b" class="row">
        



        
        <div class="col-10 offset-1 col-xxl-12 offset-xxl-0">



        
            <h3 class="article-further-reading__title main-title--decorated mb-5">
                Further reading in this issue
            </h3>
        
        <div class="multi-search"><ul class="list-unstyled"><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">April 4, 2005</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--free">Free Access</span></span></div><h3 class="article-title card__title"><a href="/doi/10.1073/pnas.0402581102" title="Olfactory expression of a single and highly variable V1r pheromone receptor-like gene in fish species" class="text-reset animation-underline">Olfactory expression of a single and highly variable V1r pheromone receptor-like gene in fish species</a></h3></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Pfister%2C+Patrick" title="Patrick Pfister">Patrick Pfister</a>,</li>
   <li class="list-inline-item"><a href="/author/Rodriguez%2C+Ivan" title="Ivan Rodriguez">Ivan Rodriguez</a>&nbsp;and</li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">April 4, 2005</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--free">Free Access</span></span></div><h3 class="article-title card__title"><a href="/doi/10.1073/pnas.0405302102" title="Parallelism of amino acid changes at the RH1 affecting spectral sensitivity among deep-water cichlids from Lakes Tanganyika and Malawi" class="text-reset animation-underline">Parallelism of amino acid changes at the RH1 affecting spectral sensitivity among deep-water cichlids from Lakes Tanganyika and Malawi</a></h3></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Sugawara%2C+Tohru" title="Tohru Sugawara">Tohru Sugawara</a>,</li>
   <li class="list-inline-item"><a href="/author/Terai%2C+Yohey" title="Yohey Terai">Yohey Terai</a>,</li>
   <li class="list-inline-item"><a href="/author/Imai%2C+Hiroo" title="Hiroo Imai">Hiroo Imai</a>,</li>
   <li class="list-inline-item"><a href="/author/Turner%2C+George+F" title="George F. Turner">George F. Turner</a>,</li>
   <li class="list-inline-item"><a href="/author/Koblm%C3%BCller%2C+Stephan" title="Stephan Koblmüller">Stephan Koblmüller</a>,</li>
   <li class="list-inline-item"><a href="/author/Sturmbauer%2C+Christian" title="Christian Sturmbauer">Christian Sturmbauer</a>,</li>
   <li class="list-inline-item"><a href="/author/Shichida%2C+Yoshinori" title="Yoshinori Shichida">Yoshinori Shichida</a>,</li>
   <li class="list-inline-item"><a href="/author/Okada%2C+Norihiro" title="Norihiro Okada">Norihiro Okada</a>&nbsp;and</li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">April 4, 2005</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--free">Free Access</span></span></div><h3 class="article-title card__title"><a href="/doi/10.1073/pnas.0405925102" title="The kinetochore protein Ndc10p is required for spindle stability and cytokinesis in yeast" class="text-reset animation-underline">The kinetochore protein Ndc10p is required for spindle stability and cytokinesis in yeast</a></h3></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Bouck%2C+David+C" title="David C. Bouck">David C. Bouck</a>,</li>
   <li class="list-inline-item"><a href="/author/Bloom%2C+Kerry+S" title="Kerry S. Bloom">Kerry S. Bloom</a>&nbsp;and</li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li></ul></div>
</div>

        </div>
    
</div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="84483e42-7aa7-4f23-bfde-4fd1a78b5421" class="multi-search--grid py-2_5x">
        



        
        <div class="container-fluid">









        <div data-widget-def="ux3-layout-widget" data-widget-id="e2b5e288-a822-4d07-a1d2-8d6d0ede93cd" class="row">
        



        
        <div class="col-10 offset-1 col-xxl-12 offset-xxl-0">



        
        <ul role="tablist" class="nav nav-tabs"><li role="presentation" class="nav-item"><a id="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-read-tab" data-toggle="tab" href="#nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-read-pane" role="tab" aria-controls="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-read-pane" aria-selected="true" class="nav-link  active"><span>Most Read</span></a></li><li role="presentation" class="nav-item"><a id="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-cited-tab" data-toggle="tab" href="#nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-cited-pane" role="tab" aria-controls="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-cited-pane" aria-selected="false" class="nav-link"><span>Most Cited</span></a></li></ul><div class="tab-content"><div id="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-read-pane" aria-labelledby="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-read-tab" role="tabpanel" class="tab-pane fade show active">



        
        <div class="top-content"><ul class="list-unstyled"><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">December 30, 2013</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--open">Open Access</span></span></div><h3 class="article-title card__title"><a href="/doi/full/10.1073/pnas.1321664111" title="Bodily maps of emotions" class="text-reset animation-underline">Bodily maps of emotions</a></h3></div><div class="card-body card__abstract"><div class="card-body card__abstract">Emotions coordinate our behavior and physiological states during survival-salient
   events and pleasurable interactions. Even though we are often consciously aware of
   our current emotional state, such as anger or happiness, the mechanisms giving ...Emotions are often felt in the body, and somatosensory feedback has been proposed
   to trigger conscious emotional experiences. Here we reveal maps of bodily sensations
   associated with different emotions using a unique topographical self-report method.
   In ...<i></i></div></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Nummenmaa%2C+Lauri" title="Lauri Nummenmaa">Lauri Nummenmaa</a>,</li>
   <li class="list-inline-item"><a href="/author/Glerean%2C+Enrico" title="Enrico Glerean">Enrico Glerean</a>,</li>
   <li class="list-inline-item"><a href="/author/Hari%2C+Riitta" title="Riitta Hari">Riitta Hari</a>&nbsp;and</li>
   <li class="list-inline-item"><a href="/author/Hietanen%2C+Jari+K" title="Jari K. Hietanen">Jari K. Hietanen</a></li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">August 14, 2017</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--open">Open Access</span></span></div><h3 class="article-title card__title"><a href="/doi/full/10.1073/pnas.1705853114" title="Oxytocin-enforced norm compliance reduces xenophobic outgroup rejection" class="text-reset animation-underline">Oxytocin-enforced norm compliance reduces xenophobic outgroup rejection</a></h3></div><div class="card-body card__abstract"><div class="card-body card__abstract">In the midst of rapid globalization, the peaceful coexistence of cultures requires
   a deeper understanding of the forces that compel prosocial behavior and thwart xenophobia.
   Yet, the conditions promoting such outgroup-directed altruism have ...Never before have individuals had to adapt to social environments defined by such
   magnitudes of ethnic diversity and cultural differentiation. However, neurobiological
   evidence informing about strategies to reduce xenophobic sentiment and foster ...</div></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Marsh%2C+Nina" title="Nina Marsh">Nina Marsh</a>,</li>
   <li class="list-inline-item"><a href="/author/Scheele%2C+Dirk" title="Dirk Scheele">Dirk Scheele</a>,</li>
   <li class="list-inline-item"><a href="/author/Feinstein%2C+Justin+S" title="Justin S. Feinstein">Justin S. Feinstein</a>,</li>
   <li class="list-inline-item"><a href="/author/Gerhardt%2C+Holger" title="Holger Gerhardt">Holger Gerhardt</a>,</li>
   <li class="list-inline-item"><a href="/author/Strang%2C+Sabrina" title="Sabrina Strang">Sabrina Strang</a>,</li>
   <li class="list-inline-item"><a href="/author/Maier%2C+Wolfgang" title="Wolfgang Maier">Wolfgang Maier</a>&nbsp;and</li>
   <li class="list-inline-item"><a href="/author/Hurlemann%2C+Ren%C3%A9" title="René Hurlemann">René Hurlemann</a></li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">February 25, 2022</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--open">Open Access</span></span></div><h3 class="article-title card__title"><a href="/doi/full/10.1073/pnas.2122090119" title="Lack of association between pandemic chilblains and SARS-CoV-2 infection" class="text-reset animation-underline">Lack of association between pandemic chilblains and SARS-CoV-2 infection</a></h3></div><div class="card-body card__abstract"><div class="card-body card__abstract">Chilblain diagnoses have increased during the severe acute respiratory syndrome coronavirus
   2 (SARS-CoV-2) pandemic and have been attributed to viral infection and a subsequent
   robust antiviral immune response. As a result, providers have ...An increased incidence of chilblains has been observed during the severe acute respiratory
   syndrome coronavirus 2 (SARS-CoV-2) pandemic and attributed to viral infection. Direct
   evidence of this relationship has been limited, however, as most cases do not ...</div></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Gehlhausen%2C+Jeff+R" title="Jeff R. Gehlhausen">Jeff R. Gehlhausen</a>,</li>
   <li class="list-inline-item"><a href="/author/Little%2C+Alicia+J" title="Alicia J. Little">Alicia J. Little</a>,</li>
   <li class="list-inline-item"><a href="/author/Ko%2C+Christine+J" title="Christine J. Ko">Christine J. Ko</a>,</li>
   <li class="list-inline-item"><a href="/author/Emmenegger%2C+Marc" title="Marc Emmenegger">Marc Emmenegger</a>,</li>
   <li class="list-inline-item"><a href="/author/Lucas%2C+Carolina" title="Carolina Lucas">Carolina Lucas</a>,</li>
   <li class="list-inline-item"><a href="/author/Wong%2C+Patrick" title="Patrick Wong">Patrick Wong</a>,</li>
   <li class="list-inline-item"><a href="/author/Klein%2C+Jon" title="Jon Klein">Jon Klein</a>,</li>
   <li class="list-inline-item"><a href="/author/Lu%2C+Peiwen" title="Peiwen Lu">Peiwen Lu</a>,</li>
   <li class="list-inline-item"><a href="/author/Mao%2C+Tianyang" title="Tianyang Mao">Tianyang Mao</a>,</li>
   <li class="list-inline-item"><a href="/author/Jaycox%2C+Jillian" title="Jillian Jaycox">Jillian Jaycox</a>,</li>
   <li class="list-inline-item"><a href="/author/Wang%2C+Eric" title="Eric Wang">Eric Wang</a>,</li>
   <li class="list-inline-item"><a href="/author/" title=""></a>,</li>
   <li class="list-inline-item"><a href="/author/Ugwu%2C+Nelson" title="Nelson Ugwu">Nelson Ugwu</a>,</li>
   <li class="list-inline-item"><a href="/author/Muenker%2C+Cate" title="Cate Muenker">Cate Muenker</a>,</li>
   <li class="list-inline-item"><a href="/author/Mekael%2C+Dilgash" title="Dilgash Mekael">Dilgash Mekael</a>,</li>
   <li class="list-inline-item"><a href="/author/Klein%2C+Rhonda+Q" title="Rhonda Q. Klein">Rhonda Q. Klein</a>,</li>
   <li class="list-inline-item"><a href="/author/Patrignelli%2C+Robert" title="Robert Patrignelli">Robert Patrignelli</a>,</li>
   <li class="list-inline-item"><a href="/author/Antaya%2C+Richard" title="Richard Antaya">Richard Antaya</a>,</li>
   <li class="list-inline-item"><a href="/author/McNiff%2C+Jennifer" title="Jennifer McNiff">Jennifer McNiff</a>,</li>
   <li class="list-inline-item"><a href="/author/Damsky%2C+William" title="William Damsky">William Damsky</a>,</li>
   <li class="list-inline-item"><a href="/author/Kamath%2C+Kathy" title="Kathy Kamath">Kathy Kamath</a>,</li>
   <li class="list-inline-item"><a href="/author/Shon%2C+John" title="John Shon">John Shon</a>,</li>
   <li class="list-inline-item"><a href="/author/Ring%2C+Aaron+M" title="Aaron M. Ring">Aaron M. Ring</a>,</li>
   <li class="list-inline-item"><a href="/author/Yildirim%2C+Inci" title="Inci Yildirim">Inci Yildirim</a>,</li>
   <li class="list-inline-item"><a href="/author/Omer%2C+Saad" title="Saad Omer">Saad Omer</a>,</li>
   <li class="list-inline-item"><a href="/author/Ko%2C+Albert+I" title="Albert I. Ko">Albert I. Ko</a>,</li>
   <li class="list-inline-item"><a href="/author/Aguzzi%2C+Adriano" title="Adriano Aguzzi">Adriano Aguzzi</a>,</li>
   <li class="list-inline-item"><a href="/author/Iwasaki%2C+Akiko" title="Akiko Iwasaki">Akiko Iwasaki</a>,</li>
   <li class="list-inline-item"><a href="/author/Obaid%2C+Abeer" title="Abeer Obaid">Abeer Obaid</a>,</li>
   <li class="list-inline-item"><a href="/author/Lu-Culligan%2C+Alice" title="Alice Lu-Culligan">Alice Lu-Culligan</a>,</li>
   <li class="list-inline-item"><a href="/author/Nelson%2C+Allison" title="Allison Nelson">Allison Nelson</a>,</li>
   <li class="list-inline-item"><a href="/author/Brito%2C+Anderson" title="Anderson Brito">Anderson Brito</a>,</li>
   <li class="list-inline-item"><a href="/author/Nunez%2C+Angela" title="Angela Nunez">Angela Nunez</a>,</li>
   <li class="list-inline-item"><a href="/author/Martin%2C+Anjelica" title="Anjelica Martin">Anjelica Martin</a>,</li>
   <li class="list-inline-item"><a href="/author/Watkins%2C+Annie" title="Annie Watkins">Annie Watkins</a>,</li>
   <li class="list-inline-item"><a href="/author/Geng%2C+Bertie" title="Bertie Geng">Bertie Geng</a>,</li>
   <li class="list-inline-item"><a href="/author/Kalinich%2C+Chaney" title="Chaney Kalinich">Chaney Kalinich</a>,</li>
   <li class="list-inline-item"><a href="/author/Harden%2C+Christina" title="Christina Harden">Christina Harden</a>,</li>
   <li class="list-inline-item"><a href="/author/Todeasa%2C+Codruta" title="Codruta Todeasa">Codruta Todeasa</a>,</li>
   <li class="list-inline-item"><a href="/author/Jensen%2C+Cole" title="Cole Jensen">Cole Jensen</a>,</li>
   <li class="list-inline-item"><a href="/author/Kim%2C+Daniel" title="Daniel Kim">Daniel Kim</a>,</li>
   <li class="list-inline-item"><a href="/author/McDonald%2C+David" title="David McDonald">David McDonald</a>,</li>
   <li class="list-inline-item"><a href="/author/Shepard%2C+Denise" title="Denise Shepard">Denise Shepard</a>,</li>
   <li class="list-inline-item"><a href="/author/Courchaine%2C+Edward" title="Edward Courchaine">Edward Courchaine</a>,</li>
   <li class="list-inline-item"><a href="/author/White%2C+Elizabeth+B" title="Elizabeth B. White">Elizabeth B. White</a>,</li>
   <li class="list-inline-item"><a href="/author/Song%2C+Eric" title="Eric Song">Eric Song</a>,</li>
   <li class="list-inline-item"><a href="/author/Silva%2C+Erin" title="Erin Silva">Erin Silva</a>,</li>
   <li class="list-inline-item"><a href="/author/Kudo%2C+Eriko" title="Eriko Kudo">Eriko Kudo</a>,</li>
   <li class="list-inline-item"><a href="/author/DeIuliis%2C+Giuseppe" title="Giuseppe DeIuliis">Giuseppe DeIuliis</a>,</li>
   <li class="list-inline-item"><a href="/author/Rahming%2C+Harold" title="Harold Rahming">Harold Rahming</a>,</li>
   <li class="list-inline-item"><a href="/author/Park%2C+Hong-Jai" title="Hong-Jai Park">Hong-Jai Park</a>,</li>
   <li class="list-inline-item"><a href="/author/Matos%2C+Irene" title="Irene Matos">Irene Matos</a>,</li>
   <li class="list-inline-item"><a href="/author/Nouws%2C+Jessica" title="Jessica Nouws">Jessica Nouws</a>,</li>
   <li class="list-inline-item"><a href="/author/Valdez%2C+Jordan" title="Jordan Valdez">Jordan Valdez</a>,</li>
   <li class="list-inline-item"><a href="/author/Fauver%2C+Joseph" title="Joseph Fauver">Joseph Fauver</a>,</li>
   <li class="list-inline-item"><a href="/author/Lim%2C+Joseph" title="Joseph Lim">Joseph Lim</a>,</li>
   <li class="list-inline-item"><a href="/author/Rose%2C+Kadi-Ann" title="Kadi-Ann Rose">Kadi-Ann Rose</a>,</li>
   <li class="list-inline-item"><a href="/author/Anastasio%2C+Kelly" title="Kelly Anastasio">Kelly Anastasio</a>,</li>
   <li class="list-inline-item"><a href="/author/Brower%2C+Kristina" title="Kristina Brower">Kristina Brower</a>,</li>
   <li class="list-inline-item"><a href="/author/Glick%2C+Laura" title="Laura Glick">Laura Glick</a>,</li>
   <li class="list-inline-item"><a href="/author/Sharma%2C+Lokesh" title="Lokesh Sharma">Lokesh Sharma</a>,</li>
   <li class="list-inline-item"><a href="/author/Sewanan%2C+Lorenzo" title="Lorenzo Sewanan">Lorenzo Sewanan</a>,</li>
   <li class="list-inline-item"><a href="/author/Knaggs%2C+Lynda" title="Lynda Knaggs">Lynda Knaggs</a>,</li>
   <li class="list-inline-item"><a href="/author/Minasyan%2C+Maksym" title="Maksym Minasyan">Maksym Minasyan</a>,</li>
   <li class="list-inline-item"><a href="/author/Batsu%2C+Maria" title="Maria Batsu">Maria Batsu</a>,</li>
   <li class="list-inline-item"><a href="/author/Petrone%2C+Mary" title="Mary Petrone">Mary Petrone</a>,</li>
   <li class="list-inline-item"><a href="/author/Kuang%2C+Maxine" title="Maxine Kuang">Maxine Kuang</a>,</li>
   <li class="list-inline-item"><a href="/author/Nakahata%2C+Maura" title="Maura Nakahata">Maura Nakahata</a>,</li>
   <li class="list-inline-item"><a href="/author/Campbell%2C+Melissa" title="Melissa Campbell">Melissa Campbell</a>,</li>
   <li class="list-inline-item"><a href="/author/Linehan%2C+Melissa" title="Melissa Linehan">Melissa Linehan</a>,</li>
   <li class="list-inline-item"><a href="/author/Askenase%2C+Michael+H" title="Michael H. Askenase">Michael H. Askenase</a>,</li>
   <li class="list-inline-item"><a href="/author/Simonov%2C+Michael" title="Michael Simonov">Michael Simonov</a>,</li>
   <li class="list-inline-item"><a href="/author/Smolgovsky%2C+Mikhail" title="Mikhail Smolgovsky">Mikhail Smolgovsky</a>,</li>
   <li class="list-inline-item"><a href="/author/Sonnert%2C+Nicole" title="Nicole Sonnert">Nicole Sonnert</a>,</li>
   <li class="list-inline-item"><a href="/author/Naushad%2C+Nida" title="Nida Naushad">Nida Naushad</a>,</li>
   <li class="list-inline-item"><a href="/author/Vijayakumar%2C+Pavithra" title="Pavithra Vijayakumar">Pavithra Vijayakumar</a>,</li>
   <li class="list-inline-item"><a href="/author/Martinello%2C+Rick" title="Rick Martinello">Rick Martinello</a>,</li>
   <li class="list-inline-item"><a href="/author/Datta%2C+Rupak" title="Rupak Datta">Rupak Datta</a>,</li>
   <li class="list-inline-item"><a href="/author/Handoko%2C+Ryan" title="Ryan Handoko">Ryan Handoko</a>,</li>
   <li class="list-inline-item"><a href="/author/Bermejo%2C+Santos" title="Santos Bermejo">Santos Bermejo</a>,</li>
   <li class="list-inline-item"><a href="/author/Prophet%2C+Sarah" title="Sarah Prophet">Sarah Prophet</a>,</li>
   <li class="list-inline-item"><a href="/author/Bickerton%2C+Sean" title="Sean Bickerton">Sean Bickerton</a>,</li>
   <li class="list-inline-item"><a href="/author/Velazquez%2C+Sofia" title="Sofia Velazquez">Sofia Velazquez</a>,</li>
   <li class="list-inline-item"><a href="/author/Alpert%2C+Tara" title="Tara Alpert">Tara Alpert</a>,</li>
   <li class="list-inline-item"><a href="/author/Rice%2C+Tyler" title="Tyler Rice">Tyler Rice</a>,</li>
   <li class="list-inline-item"><a href="/author/Khoury-Hanold%2C+William" title="William Khoury-Hanold">William Khoury-Hanold</a>,</li>
   <li class="list-inline-item"><a href="/author/Peng%2C+Xiaohua" title="Xiaohua Peng">Xiaohua Peng</a>,</li>
   <li class="list-inline-item"><a href="/author/Yang%2C+Yexin" title="Yexin Yang">Yexin Yang</a>,</li>
   <li class="list-inline-item"><a href="/author/Cao%2C+Yiyun" title="Yiyun Cao">Yiyun Cao</a>&nbsp;and</li>
   <li class="list-inline-item"><a href="/author/Strong%2C+Yvette" title="Yvette Strong">Yvette Strong</a></li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li></ul></div>
</div><div id="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-cited-pane" aria-labelledby="nav-d25abac7-d1cf-406b-b915-0c426445c9d1-most-cited-tab" role="tabpanel" class="tab-pane fade">



        
        <div class="top-content"><ul class="list-unstyled"><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">September 30, 2005</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--free">Free Access</span></span></div><h3 class="article-title card__title"><a href="/doi/full/10.1073/pnas.0506580102" title="Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles" class="text-reset animation-underline">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</a></h3></div><div class="card-body card__abstract"><div class="card-body card__abstract">
   Although genomewide RNA expression analysis has become a routine tool in biomedical
   research, extracting biological insight from such information remains a major challenge.
   Here, we describe a powerful analytical method called Gene Set Enrichment Analysis
   ...</div></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Subramanian%2C+Aravind" title="Aravind Subramanian">Aravind Subramanian</a>,</li>
   <li class="list-inline-item"><a href="/author/Tamayo%2C+Pablo" title="Pablo Tamayo">Pablo Tamayo</a>,</li>
   <li class="list-inline-item"><a href="/author/Mootha%2C+Vamsi+K" title="Vamsi K. Mootha">Vamsi K. Mootha</a>,</li>
   <li class="list-inline-item"><a href="/author/Mukherjee%2C+Sayan" title="Sayan Mukherjee">Sayan Mukherjee</a>,</li>
   <li class="list-inline-item"><a href="/author/Ebert%2C+Benjamin+L" title="Benjamin L. Ebert">Benjamin L. Ebert</a>,</li>
   <li class="list-inline-item"><a href="/author/Gillette%2C+Michael+A" title="Michael A. Gillette">Michael A. Gillette</a>,</li>
   <li class="list-inline-item"><a href="/author/Paulovich%2C+Amanda" title="Amanda Paulovich">Amanda Paulovich</a>,</li>
   <li class="list-inline-item"><a href="/author/Pomeroy%2C+Scott+L" title="Scott L. Pomeroy">Scott L. Pomeroy</a>,</li>
   <li class="list-inline-item"><a href="/author/Golub%2C+Todd+R" title="Todd R. Golub">Todd R. Golub</a>,</li>
   <li class="list-inline-item"><a href="/author/Lander%2C+Eric+S" title="Eric S. Lander">Eric S. Lander</a>&nbsp;and</li>
   <li class="list-inline-item"><a href="/author/Mesirov%2C+Jill+P" title="Jill P. Mesirov">Jill P. Mesirov</a></li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">June 3, 2010</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--free">Free Access</span></span></div><h3 class="article-title card__title"><a href="/doi/full/10.1073/pnas.1000080107" title="Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample" class="text-reset animation-underline">Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample</a></h3></div><div class="card-body card__abstract"><div class="card-body card__abstract">The ongoing revolution in high-throughput sequencing continues to democratize the
   ability of small groups of investigators to map the microbial component of the biosphere.
   In particular, the coevolution of new sequencing platforms and new software tools
   ...</div></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Caporaso%2C+J+Gregory" title="J. Gregory Caporaso">J. Gregory Caporaso</a>,</li>
   <li class="list-inline-item"><a href="/author/Lauber%2C+Christian+L" title="Christian L. Lauber">Christian L. Lauber</a>,</li>
   <li class="list-inline-item"><a href="/author/Walters%2C+William+A" title="William A. Walters">William A. Walters</a>,</li>
   <li class="list-inline-item"><a href="/author/Berg-Lyons%2C+Donna" title="Donna Berg-Lyons">Donna Berg-Lyons</a>,</li>
   <li class="list-inline-item"><a href="/author/Lozupone%2C+Catherine+A" title="Catherine A. Lozupone">Catherine A. Lozupone</a>,</li>
   <li class="list-inline-item"><a href="/author/Turnbaugh%2C+Peter+J" title="Peter J. Turnbaugh">Peter J. Turnbaugh</a>,</li>
   <li class="list-inline-item"><a href="/author/Fierer%2C+Noah" title="Noah Fierer">Noah Fierer</a>&nbsp;and</li>
   <li class="list-inline-item"><a href="/author/Knight%2C+Rob" title="Rob Knight">Rob Knight</a></li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li><li><div class="card    "><div class="card-content"><div class="card-header card__header mb-2"><div class="d-flex align-items-center mb-2"><span class="overline card__meta text-uppercase"><a href="/topic/type/research-article" class="card__meta__type">Research Article</a><span class="pl-2 ml-2 pl-2 border-left border-darker-gray  card__meta__date">June 11, 2002</span><span class="pl-2 ml-2 border-left border-darker-gray  card__meta__access meta__access meta__access--free">Free Access</span></span></div><h3 class="article-title card__title"><a href="/doi/full/10.1073/pnas.122653799" title="Community structure in social and biological networks" class="text-reset animation-underline">Community structure in social and biological networks</a></h3></div><div class="card-body card__abstract"><div class="card-body card__abstract">
   A number of recent studies have focused on the statistical properties of networked
   systems such as social networks and the Worldwide Web. Researchers have concentrated
   particularly on a few properties that seem to be common to many networks: the ...</div></div><div data-visible-items-sm="2" data-visible-items-md="2" data-visible-items="2" data-truncate-less="less" data-truncate-dots="true" class="card-contribs authors text-reset authors-truncate"><ul class="list-inline" title="list of authors">
   <li class="list-inline-item"><a href="/author/Girvan%2C+M" title="M. Girvan">M. Girvan</a>&nbsp;and</li>
   <li class="list-inline-item"><a href="/author/Newman%2C+M+E+J" title="M. E. J. Newman">M. E. J. Newman</a></li>
</ul>
</div><div id="abs-section-id" class="card__extra-info collapse"><span class="card__extra-info__relation">This is an addendum to</span><a href="#" class="card__extra-info__link">Complex viscosity of helical and doubly helical polymeric liquids from general rigid bead-rod theory</a></div></div></div></li></ul></div>
</div></div>
</div>

        </div>
    
</div>

        </div>
    




        
        <section class="mt-2x">



        
        <div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=0b3c9833-2203-446a-b8eb-6917fc2ad6a3&amp;doi=10.1073%2Fpnas.0501761102&amp;pbContext=%3Bjournal%3Ajournal%3Apnas%3Bpage%3Astring%3AArticle%2FChapter+View%3BrequestedJournal%3Ajournal%3Apnas%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apnas-site%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1073%2Fpnas.0501761102%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3Adoi%5C%3A10.1073%2Fpnas.2005.102.issue-15" class="show-recommended-placeholder"></div>
</section>
</main>




        
        









        <section data-widget-def="ux3-layout-widget" data-widget-id="a20ca6a1-603b-499e-81db-ee1dc31f5224" class="signup-ad" aria-label="signup-ad">
        



        
        <div class="signup-ad__container">



        
        <!-- Begin Mailchimp Signup Form -->
<h3 class="signup-ad__title">Sign up for the<span class="d-block">PNAS Highlights newsletter</span></h3>
<div class="signup-ad__content">Get in-depth science stories sent to your inbox twice a month.</div>

<form action="https://pnas.us15.list-manage.com/subscribe/post?u=371d96023d24fb109b778621f&amp;id=b053c2ccce" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate signup-ad__form" target="_blank" novalidate>
                <input type="email" value="" name="EMAIL" class="signup-ad__input" id="mce-EMAIL" placeholder="name@example.com" required>

    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->

     <input style="display:none;" type="text" name="b_371d96023d24fb109b778621f_b053c2ccce" tabindex="-1" value="" aria-hidden="true">
    <button type="submit" name="subscribe" id="mc-embedded-subscribe" class="signup-ad__submit">Subscribe<i aria-hidden="true" class="icon-arrow-right"></i></button>
</form>

<!--End mc_embed_signup -->
</div>

        </section>
    
<footer class="footer pb-3"><section class="footer__top py-3"><div class="container-fluid"><div class="row align-items-center"><div class=" col-sm-6 col-md-4 my-4 d-flex justify-content-center flex-column">



        
        <a href="/" title="PNAS logo" class="footer__logo-wrapper">
    <img src="/pb-assets/images/Logos/footer/logo-footer-1624408444100.svg
" alt="PNAS logo" class="footer__logo">
</a>




        
        <div class="footer__social-media__wrapper d-flex mt-3"><div class="footer__social-media">
    <ul role="list" class="list-inline footer__social-media__list text-center d-flex flex-wrap">
        <li role="listitem" class="list-inline-item footer__social-media__list-item"><a href="https://www.facebook.com/pages/PNAS/18262365099" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-facebook"></i></a></li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item"><a href="https://twitter.com/PNASNews" title="Twitter" target="_blank"><i aria-hidden="true" class="icon-twitter"></i></a></li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item"><a href="https://www.youtube.com/channel/UCYjvFEQMgH6ou8fSYfiOKCw" title="Youtube" target="_blank"><i aria-hidden="true" class="icon-youtube"></i></a></li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item"><a href="/action/showPreferences?menuTab=Alerts" title="Mail"><i aria-hidden="true" class="icon-mail"></i></a></li>
        <li role="listitem" class="list-inline-item footer__social-media__list-item"><a href="/about/rss" title="RSS feeds"><i aria-hidden="true" class="icon-rss"></i></a></li>
    </ul>
    </div>
</div>
</div><div class=" col-sm-6 col-md-4 my-4 text-uppercase font-weight-bold">



        
        <style>
    .headerlist { 
            font-size: 17px;
            font-weight: bold;
            text-align: left; 
            margin-bottom: .7rem;
        }
    .footer__sitemap__list-item { 
            text-align: left;
            font-weight: 400;
        }
    </style>
    <div class="footer__sitemap">
        <div class="headerlist">Browse</div>
        <ul role="list" class="list-unstyled footer__sitemap__list text-center">
            <li role="listitem" class="footer__sitemap__list-item">
                <a href="/toc/pnas/current" title="Current Issue" class="animation-underline">Current Issue</a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="https://academic.oup.com/pnasnexus" title="PNAS Nexus" class="animation-underline"><em>PNAS Nexus</em></a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="/about/special-features" title="List of Issues" class="animation-underline">Special Features</a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="/about/colloquia" title="List of Issues" class="animation-underline">Colloquia</a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="/loi/pnas" title="List of Issues" class="animation-underline">List of Issues</a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="/about/collected-papers" title="Collected Papers" class="animation-underline">Collected Papers</a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="/front-matter" title="Front Matter" class="animation-underline">Front Matter</a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="/journal-club" title="Journal Club" class="animation-underline">Journal Club</a>
            </li>
            <li role="listitem" class="footer__sitemap__list-item my-2">
                <a href="/about/science-sessions-podcast" title="Podcasts" class="animation-underline">Podcasts</a>
            </li>
        </ul>
    </div>
</div><div class=" col-sm-6 col-md-4 my-4 text-uppercase font-weight-bold">



        
        <style>
        .headerlist { 
                font-size: 17px;
                font-weight: bold;
                text-align: left; 
                margin-bottom: .7rem;
            },
            .footer__sitemap__list-item { 
                    text-align: left;
                    font-weight: 400;
                }
        </style>
        <div class="footer__sitemap">
            <div class="headerlist">Information</div>
            <ul role="list" class="list-unstyled footer__sitemap__list text-center">
                <li role="listitem" class="footer__sitemap__list-item">
                    <a href="/about" title="About" class="animation-underline">About</a>
                </li>
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/about/diversity-inclusion-pnas" title="Diversity and Inclusion" class="animation-underline">Diversity and Inclusion</a>
                </li>
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/author-center" title="Authors" class="animation-underline">Authors</a>
                </li>
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/reviewer" title="Reviewers" class="animation-underline">Reviewers</a>
                </li>
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/subscriptions" title="Suscribers" class="animation-underline">Subscribers</a>
                </li>
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/librarians" title="Librarians" class="animation-underline">Librarians</a>
                </li>
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/about/press" title="Press" class="animation-underline">Press</a>
                </li>		
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/about/cozzarelli-prize" title="Cozzarelli Prize" class="animation-underline">Cozzarelli Prize</a>
                </li>
                <li role="listitem" class="footer__sitemap__list-item my-2">
                    <a href="/updates" title="PNAS Updates" class="animation-underline">PNAS Updates</a>
                </li>
            </ul>
        </div>
</div></div></div><div data-show-after="400" class="back-to-top back-to-top--align-right  "><button data-snap="footer__top" class="back-to-top__action position-relative p-0 m-0 border-0 d-flex flex-column align-items-center justify-content-center"><svg viewBox="0 0 36 36" class="position-relative"><path d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" class="back-to-top__action__ring-bg"></path><path d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831" stroke-dasharray="0, 100" class="back-to-top__action__ring"></path></svg><span class="back-to-top__action-icon position-absolute d-flex"><i aria-hidden="true" class="icon-arrow-up"></i></span><span class="sr-only">Back to top</span></button></div></section><section class="footer__bottom"><div class="container-100"><div class="row align-items-center"><div class="col-md-8 col-sm-10">



        
        <div class="footer__copyright text-center text-sm-left px-sm-0 px-4"><p class="m-0">Copyright © 2022 National Academy of Science. All rights reserved. | Online ISSN 1091-6490</p><p class="m-0">PNAS is a partner of CHORUS, CLOCKSS, COPE, CrossRef, ORCID, and Research4Life.</p></div>
</div><div class="col-md-4 col-sm-2 text-sm-right text-center mt-sm-0 mt-3">



        
        <ul role="list" class="list-inline footer__copyright__links">
    <li role="listitem" class="list-inline-item">
        <a href="/about/staff-directory" title="Contact">Contact</a>
	</li>
	<li role="listitem" class="list-inline-item">
        <a href="/about/site-map" title="Site Map">Site Map</a>
	</li>
	<li role="listitem" class="list-inline-item">
        <a href="/about/terms" title="Terms">Terms &amp; Privacy Policy</a>
	</li>
        <li class="list-inline-item">
        <a href="/about/accessibility" title="Accessibility">Accessibility</a>
    </li>
</ul>
</div></div></div></section></footer>




        
        




<div class="modal fade" role="dialog" tabindex="-1" id="modal--requestUsername">
    <div class="modal-dialog modal-fullscreen-sm-down" role="document">
        <div class="modal-content content">
            <div class="modal-header">
                <h2 class="modal-title h5">Request Username</h2>
                <button type="button" data-dismiss="modal" aria-label="close" class="close">
                    <span aria-hidden="true">x</span>
                    <span class="sr-only">Close</span>
                </button>
            </div>
            <div class="modal-body">
                <form action="/action/requestUsername" class="raa-form" method="post"><input type="hidden" name="requestUsername" value="requestUsername" />
                    <p>Can't sign in? Forgot your username? <br />
                        Enter your email address below and we will send you your username</p>
                    <div class="alert alert-danger d-none"></div>

                    <div class="form-row">
                        <div class="col">
                            <div class="form-group">
                                <label for="6eccc267-4bd8-4882-88e8-d9b06094352c.email">Email</label>
                                <input id="6eccc267-4bd8-4882-88e8-d9b06094352c.email" class="form-control" type="text" name="email" value="" size="15" />
                            </div>
                        </div>
                    </div>
                    
                        <div class="form-row">
                            <div class="col">
                                <div class="form-group">
                                    <div class="username-recaptcha-ajax"></div>
                                </div>
                            </div>
                        </div>
                    

                    <div class="form-row">
                        <div class="col text-right">
                            <button type="submit" data-dismiss="modal" class="btn btn-submit">
                                <span>Submit</span>
                                <span aria-hidden="true" class="icon-arrow-right ml-3"></span>
                            </button>
                        </div>
                    </div></form>

                <div class="alert alert-success d-none">
                    <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
                </div>

            </div>
        </div>
    </div>
</div>




        
        





<div tabindex="-1" id="modal--registration" aria-labelledby="modalRegistrationLabel" class="modal fade" aria-modal="true" role="dialog">
    <div role="document" class="modal-dialog modal-fullscreen-sm-down">
        <div class="modal-content content">
            <div class="modal-header">
                <h2 id="modalRegistrationLabel" class="modal-title h5">Create a new account</h2>
                <button type="button" data-dismiss="modal" aria-label="close" class="close">
                    <span aria-hidden="true">x</span><span class="sr-only">Close</span>
                </button>
            </div>
            <div class="modal-body">
                <form action="/action/registration" class="registration-form raa-form" method="post"><input type="hidden" name="redirectUri" value="/doi/10.1073/pnas.0501761102" />
                    <div class="form-row">
                        <div class="col">
                            <div class="form-group">
                                <label for="f96c9a0d-f4a9-4ca3-a52b-99fb097c56d6.email">Email</label>
                                <input id="f96c9a0d-f4a9-4ca3-a52b-99fb097c56d6.email" class="email form-control" type="email" name="email" value=""required="required" />
                            </div>
                        </div>
                    </div>
                    <div class="form-row">
                        <div class="col-12 col-sm-6">
                            <a href="/action/showLogin" class="show-login">
                                
                                Returning user
                            </a>
                        </div>
                        <div class="col text-right">
                            <button type="submit" data-dismiss="modal" class="btn btn-submit">
                                <span>Submit</span>
                                <span aria-hidden="true" class="icon-arrow-right ml-3"></span>
                            </button>
                        </div>
                    </div></form>
            </div>
        </div>
    </div>
</div>




        
        <aside class="cookie-policy"> <p>We use cookies on this site to enhance your user experience. By using this website, you are giving your consent for us to set cookies. <a href="/help/cookie-policy">Find out more</a></p><a href="/help/cookie-policy" class="btn btn-primary">CONTINUE</a></aside>




        
        














<div tabindex="-1" id="modal--login" aria-labelledby="modalLoginLabel" class="modal fade" aria-modal="true" role="dialog">
    <div role="document" class="modal-dialog modal-fullscreen-sm-down">
        <div class="modal-content content">
            <div class="modal-header">
                <h2 id="modalLabelLogin" class="modal-title h5">Login</h2>
                <button type="button" data-dismiss="modal" aria-label="close" class="close">
                    <span aria-hidden="true">x</span><span class="sr-only">Close</span>
                </button>
            </div>
            




















<div class="modal-body">
    <form action="/action/doLogin" id="loginForm" class="raa-form login-form " method="post"><input type="hidden" name="id" value="465aa1e9-3090-4f74-8ca8-f0d260f07eeb" />
        





        <input type="hidden" name="redirectUri" value="/doi/10.1073/pnas.0501761102" />
    

<input type="hidden" name="loginUri" value="/doi/10.1073/pnas.0501761102" />
        
            <input type="hidden" name="popup" value="true" />
        
        
                <div class="form-row">
                    <div class="col-12 col-sm-6">
                        <div class="form-group">
                            

























    
        
    
    
        
    
    
    
        
    

        

        
    <div class="form-group">
        <div class="label ">

            
                    <label for="login">Email<span class="required-marker" aria-hidden="true">*</span></label>
                
            
        </div>
        
        
        <input id="login" class="form-control" type="text" name="login" value="" size="15" placeholder="Enter your email"aria-required="true" />
        
                            
    </div>

                        </div>
                        <div class="actions raa-form__reset-password text-right mt-n4">
                            
                        </div>
                    </div>

                    <div class="col">
                        <div class="form-group">
                            





















    



    
    
    <div class="form-group">
        <div class="label ">
            <label for="password">Password<span class="required-marker"
                                                                                        aria-hidden="true">*</span></label>
            
        </div>
        
        
        <input id="password" class="form-control" type="password" name="password" value="" autocomplete="off" placeholder="Enter your password"aria-required="true" />
        <span class="password-eye-icon icon-eye hidden"></span>
        
                            
    </div>

                        </div>
                        <div class="raa-form__reset-password text-right">
                            <span>Forgot your password? Reset it </span>
                            <a href="/action/requestResetPassword?data-target=%23modal--resetPassword&amp;data-toggle=modal&amp;data-dismiss=modal" type="button" data-target="#modal--resetPassword" data-toggle="modal" data-dismiss="modal">here</a>
                        </div>
                    </div>
                </div>
            
        <div class="form-row mt-4">
            <div class="col-12 col-sm-6 text-left">
                <div class="custom-control custom-checkbox">
                    
                            <input id="465aa1e9-3090-4f74-8ca8-f0d260f07eeb-remember" class="custom-control-input" type="checkbox" name="remember" value="true" />
                        
                    <label for="465aa1e9-3090-4f74-8ca8-f0d260f07eeb-remember" class="custom-control-label">
                        Keep me logged in
                    </label>
                </div>
            </div>
            
                    <div class="col text-right">
                        <button type="submit" class="btn btn-submit">
                            <span>Login</span>
                            <span aria-hidden="true" class="icon-arrow-right ml-3"></span>
                        </button>
                    </div>
                
        </div>
        <div class="form-row border-top mt-3 pt-3">
            <div class="col raa-form__register">
                <span>Don’t have an account? Register</span>
                <a href="/action/registration">
                    here
                </a>
                
                    <a href="/action/ssostart?redirectUri=%2Fdoi%2F10.1073%2Fpnas.0501761102" class="float-right">
                        
                        <div class="institution-login">
                            Institutional Login
                        </div>
                    </a>
                
                
            </div>
        </div></form>
</div>
        </div>
    </div>
</div>




        
        




<div tabindex="-1" id="modal--verifyPhone"
     aria-labelledby="modalverifyPhoneLabel"
     class="modal fade" aria-modal="true" role="dialog">
    <div role="document" class="modal-dialog modal-fullscreen-sm-down">
        <div class="modal-content">
            <div class="modal-header">
                <h2 id="modalverifyPhoneLabel" class="modal-title h5">Change Password </h2>
                <button type="button" data-dismiss="modal" aria-label="close" class="close">
                    <span aria-hidden="true">x</span><span class="sr-only">Close</span>
                </button>
            </div>

            <div class="modal-body">
                <form action="/action/verifyPhoneNumber" method="post"><input type="hidden" name="submit" value="submit" />
                    <div class="alert alert-danger d-none"></div>
                    <div class="form-row">
                        <div class="col">
                            <div class="form-group">
                                <label for="946b33a7-d695-4878-8fb4-546789ae57b3-code">Enter the verification code</label>
                                <input id="946b33a7-d695-4878-8fb4-546789ae57b3-code" class="form-control" type="text" name="verificationCode" value="" size="15"pattern="^[0-9]{6}$"required="required" />
                            </div>
                        </div>
                    </div>
                    <div class="form-row">
                        <div class="col text-right">
                            <button type="submit" class="btn btn-submit">
                                <span>Submit</span>
                                <span aria-hidden="true" class="icon-arrow-right ml-3"></span>
                            </button>
                        </div>
                    </div></form>
            </div>

        </div>
        <div class="alert alert-success d-none">
            <h5>Congrats!</h5>
            <p>Your Phone has been verified</p>
        </div>
    </div>
</div>
</div>
        </div>
    </div>
        
        

            <script src="/products/pnas/releasedAssets/js/build.lazyload.bundle-62679fbcca1c44cbefa2.js"></script>

        <script src="/products/pnas/releasedAssets/js/main.bundle-2636f2c1676b678882ad.js" ></script>
        

            <script type="text/javascript" src="/wro/l7ul~article-metrics-phase2.js"></script>
        















<script defer src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon='{"rayId":"706f43004b4231eb","token":"4fddf4ff2db74c469c27e6f3ddde2174","version":"2021.12.0","si":100}' crossorigin="anonymous"></script>
</body>
</html>
